benchmarks/transformer/better_transformer_vs_mha_functional.py

```diff
@@ -0,0 +1,195 @@
+"""
```

Comment:
Running this on AWS cluster with a100:

|dtype        |batch_size|sequence_length|n_heads|embed_dim|time_native_mha_slow(μs)|time_native_mha_fast (μs)|speedup flash_mha v native_mha|padding|
|-------------|----------|---------------|-------|---------|------------------------|-------------------------|------------------------------|-------|
|torch.float16|16        |64             |8      |512      |399.432                 |273.695                  |1.459                         |0.078  |
|torch.float16|16        |64             |8      |512      |389.560                 |279.316                  |1.395                         |0.240  |
|torch.float16|16        |64             |8      |512      |390.574                 |263.137                  |1.484                         |0.375  |
|torch.float16|16        |64             |8      |512      |395.315                 |265.861                  |1.487                         |0.457  |
|torch.float16|16        |64             |8      |512      |388.219                 |260.526                  |1.490                         |0.510  |
|torch.float16|16        |64             |8      |512      |388.168                 |264.131                  |1.470                         |0.553  |
|torch.float16|16        |64             |8      |512      |392.468                 |261.734                  |1.499                         |0.521  |
|torch.float16|16        |64             |8      |512      |387.553                 |257.690                  |1.504                         |0.555  |
|torch.float16|16        |64             |8      |512      |390.820                 |266.015                  |1.469                         |0.580  |
|torch.float16|16        |64             |8      |512      |391.240                 |260.301                  |1.503                         |0.602  |
|torch.float16|16        |64             |8      |512      |386.263                 |259.809                  |1.487                         |0.629  |
|torch.float16|16        |64             |8      |512      |384.328                 |263.209                  |1.460                         |0.645  |
|torch.float16|16        |64             |8      |512      |389.806                 |264.827                  |1.472                         |0.662  |
|torch.float16|16        |64             |8      |512      |392.038                 |258.232                  |1.518                         |0.680  |
|torch.float16|16        |64             |8      |512      |384.133                 |264.233                  |1.454                         |0.693  |
|torch.float16|16        |64             |8      |512      |399.954                 |268.298                  |1.491                         |0.703  |
|torch.float16|16        |64             |16     |512      |396.718                 |267.377                  |1.484                         |0.078  |
|torch.float16|16        |64             |16     |512      |393.554                 |263.260                  |1.495                         |0.240  |
|torch.float16|16        |64             |16     |512      |391.076                 |263.987                  |1.481                         |0.375  |
|torch.float16|16        |64             |16     |512      |392.284                 |261.396                  |1.501                         |0.457  |
|torch.float16|16        |64             |16     |512      |387.666                 |257.925                  |1.503                         |0.510  |
|torch.float16|16        |64             |16     |512      |394.772                 |266.936                  |1.479                         |0.553  |
|torch.float16|16        |64             |16     |512      |400.097                 |266.854                  |1.499                         |0.521  |
|torch.float16|16        |64             |16     |512      |419.779                 |282.542                  |1.486                         |0.555  |
|torch.float16|16        |64             |16     |512      |400.353                 |270.561                  |1.480                         |0.580  |
|torch.float16|16        |64             |16     |512      |399.176                 |271.084                  |1.473                         |0.602  |
|torch.float16|16        |64             |16     |512      |401.388                 |270.397                  |1.484                         |0.629  |
|torch.float16|16        |64             |16     |512      |394.230                 |264.059                  |1.493                         |0.645  |
|torch.float16|16        |64             |16     |512      |399.903                 |269.425                  |1.484                         |0.662  |
|torch.float16|16        |64             |16     |512      |403.712                 |267.735                  |1.508                         |0.680  |
|torch.float16|16        |64             |16     |512      |406.405                 |265.544                  |1.530                         |0.693  |
|torch.float16|16        |64             |16     |512      |395.069                 |266.004                  |1.485                         |0.703  |
|torch.float16|16        |64             |8      |1024     |405.883                 |270.090                  |1.503                         |0.078  |
|torch.float16|16        |64             |8      |1024     |409.068                 |263.967                  |1.550                         |0.240  |
|torch.float16|16        |64             |8      |1024     |413.635                 |269.220                  |1.536                         |0.375  |
|torch.float16|16        |64             |8      |1024     |417.894                 |275.384                  |1.517                         |0.457  |
|torch.float16|16        |64             |8      |1024     |417.823                 |269.865                  |1.548                         |0.510  |
|torch.float16|16        |64             |8      |1024     |418.662                 |271.647                  |1.541                         |0.553  |
|torch.float16|16        |64             |8      |1024     |403.220                 |268.564                  |1.501                         |0.521  |
|torch.float16|16        |64             |8      |1024     |399.135                 |260.188                  |1.534                         |0.555  |
|torch.float16|16        |64             |8      |1024     |397.763                 |254.833                  |1.561                         |0.580  |
|torch.float16|16        |64             |8      |1024     |402.391                 |259.052                  |1.553                         |0.602  |
|torch.float16|16        |64             |8      |1024     |396.790                 |260.495                  |1.523                         |0.629  |
|torch.float16|16        |64             |8      |1024     |405.586                 |264.684                  |1.532                         |0.645  |
|torch.float16|16        |64             |8      |1024     |407.665                 |256.399                  |1.590                         |0.662  |
|torch.float16|16        |64             |8      |1024     |411.064                 |267.633                  |1.536                         |0.680  |
|torch.float16|16        |64             |8      |1024     |400.722                 |262.605                  |1.526                         |0.693  |
|torch.float16|16        |64             |8      |1024     |406.641                 |260.065                  |1.564                         |0.703  |
|torch.float16|16        |64             |16     |1024     |398.152                 |269.199                  |1.479                         |0.078  |
|torch.float16|16        |64             |16     |1024     |395.223                 |266.435                  |1.483                         |0.240  |
|torch.float16|16        |64             |16     |1024     |408.689                 |270.090                  |1.513                         |0.375  |
|torch.float16|16        |64             |16     |1024     |399.565                 |267.612                  |1.493                         |0.457  |
|torch.float16|16        |64             |16     |1024     |391.700                 |256.532                  |1.527                         |0.510  |
|torch.float16|16        |64             |16     |1024     |387.226                 |256.748                  |1.508                         |0.553  |
|torch.float16|16        |64             |16     |1024     |393.595                 |256.123                  |1.537                         |0.521  |
|torch.float16|16        |64             |16     |1024     |392.151                 |261.683                  |1.499                         |0.555  |
|torch.float16|16        |64             |16     |1024     |394.127                 |261.908                  |1.505                         |0.580  |
|torch.float16|16        |64             |16     |1024     |397.271                 |264.909                  |1.500                         |0.602  |
|torch.float16|16        |64             |16     |1024     |394.568                 |254.003                  |1.553                         |0.629  |
|torch.float16|16        |64             |16     |1024     |393.646                 |264.837                  |1.486                         |0.645  |
|torch.float16|16        |64             |16     |1024     |398.684                 |268.626                  |1.484                         |0.662  |
|torch.float16|16        |64             |16     |1024     |397.302                 |301.814                  |1.316                         |0.680  |
|torch.float16|16        |64             |16     |1024     |389.837                 |257.925                  |1.511                         |0.693  |
|torch.float16|16        |64             |16     |1024     |391.762                 |264.090                  |1.483                         |0.703  |
|torch.float16|16        |128            |8      |512      |446.659                 |267.889                  |1.667                         |0.077  |
|torch.float16|16        |128            |8      |512      |447.293                 |277.453                  |1.612                         |0.245  |
|torch.float16|16        |128            |8      |512      |433.449                 |266.506                  |1.626                         |0.374  |
|torch.float16|16        |128            |8      |512      |435.896                 |266.670                  |1.635                         |0.458  |
|torch.float16|16        |128            |8      |512      |429.394                 |268.677                  |1.598                         |0.508  |
|torch.float16|16        |128            |8      |512      |434.944                 |265.974                  |1.635                         |0.555  |
|torch.float16|16        |128            |8      |512      |435.313                 |277.043                  |1.571                         |0.518  |
|torch.float16|16        |128            |8      |512      |429.486                 |270.715                  |1.586                         |0.549  |
|torch.float16|16        |128            |8      |512      |436.050                 |260.434                  |1.674                         |0.579  |
|torch.float16|16        |128            |8      |512      |430.746                 |263.250                  |1.636                         |0.603  |
|torch.float16|16        |128            |8      |512      |430.735                 |260.332                  |1.655                         |0.626  |
|torch.float16|16        |128            |8      |512      |439.910                 |269.804                  |1.630                         |0.646  |
|torch.float16|16        |128            |8      |512      |439.972                 |265.718                  |1.656                         |0.661  |
|torch.float16|16        |128            |8      |512      |436.081                 |304.589                  |1.432                         |0.679  |
|torch.float16|16        |128            |8      |512      |420.588                 |265.943                  |1.581                         |0.690  |
|torch.float16|16        |128            |8      |512      |419.676                 |263.444                  |1.593                         |0.706  |
|torch.float16|16        |128            |16     |512      |430.879                 |268.851                  |1.603                         |0.077  |
|torch.float16|16        |128            |16     |512      |424.428                 |254.403                  |1.668                         |0.245  |
|torch.float16|16        |128            |16     |512      |430.756                 |256.184                  |1.681                         |0.374  |
|torch.float16|16        |128            |16     |512      |430.418                 |270.817                  |1.589                         |0.458  |
|torch.float16|16        |128            |16     |512      |431.852                 |261.765                  |1.650                         |0.508  |
|torch.float16|16        |128            |16     |512      |425.380                 |260.567                  |1.633                         |0.555  |
|torch.float16|16        |128            |16     |512      |424.069                 |267.530                  |1.585                         |0.518  |
|torch.float16|16        |128            |16     |512      |422.257                 |269.773                  |1.565                         |0.549  |
|torch.float16|16        |128            |16     |512      |425.236                 |257.085                  |1.654                         |0.579  |
|torch.float16|16        |128            |16     |512      |435.937                 |265.738                  |1.640                         |0.603  |
|torch.float16|16        |128            |16     |512      |424.059                 |267.418                  |1.586                         |0.626  |
|torch.float16|16        |128            |16     |512      |429.650                 |268.134                  |1.602                         |0.646  |
|torch.float16|16        |128            |16     |512      |430.438                 |255.078                  |1.687                         |0.661  |
|torch.float16|16        |128            |16     |512      |421.335                 |258.089                  |1.633                         |0.679  |
|torch.float16|16        |128            |16     |512      |419.379                 |260.864                  |1.608                         |0.690  |
|torch.float16|16        |128            |16     |512      |458.629                 |255.212                  |1.797                         |0.706  |
|torch.float16|16        |128            |8      |1024     |436.419                 |281.129                  |1.552                         |0.077  |
|torch.float16|16        |128            |8      |1024     |435.794                 |267.162                  |1.631                         |0.245  |
|torch.float16|16        |128            |8      |1024     |421.151                 |273.142                  |1.542                         |0.374  |
|torch.float16|16        |128            |8      |1024     |436.214                 |256.502                  |1.701                         |0.458  |
|torch.float16|16        |128            |8      |1024     |431.749                 |262.584                  |1.644                         |0.508  |
|torch.float16|16        |128            |8      |1024     |423.117                 |257.894                  |1.641                         |0.555  |
|torch.float16|16        |128            |8      |1024     |418.908                 |261.243                  |1.604                         |0.518  |
|torch.float16|16        |128            |8      |1024     |440.064                 |259.533                  |1.696                         |0.549  |
|torch.float16|16        |128            |8      |1024     |432.323                 |265.042                  |1.631                         |0.579  |
|torch.float16|16        |128            |8      |1024     |417.751                 |259.666                  |1.609                         |0.603  |
|torch.float16|16        |128            |8      |1024     |429.691                 |262.441                  |1.637                         |0.626  |
|torch.float16|16        |128            |8      |1024     |429.271                 |259.666                  |1.653                         |0.646  |
|torch.float16|16        |128            |8      |1024     |419.830                 |257.341                  |1.631                         |0.661  |
|torch.float16|16        |128            |8      |1024     |421.601                 |258.499                  |1.631                         |0.679  |
|torch.float16|16        |128            |8      |1024     |428.964                 |255.191                  |1.681                         |0.690  |
|torch.float16|16        |128            |8      |1024     |432.568                 |255.713                  |1.692                         |0.706  |
|torch.float16|16        |128            |16     |1024     |420.557                 |267.510                  |1.572                         |0.077  |
|torch.float16|16        |128            |16     |1024     |430.889                 |264.161                  |1.631                         |0.245  |
|torch.float16|16        |128            |16     |1024     |426.332                 |266.772                  |1.598                         |0.374  |
|torch.float16|16        |128            |16     |1024     |430.039                 |258.632                  |1.663                         |0.458  |
|torch.float16|16        |128            |16     |1024     |419.133                 |256.758                  |1.632                         |0.508  |
|torch.float16|16        |128            |16     |1024     |421.724                 |264.059                  |1.597                         |0.555  |
|torch.float16|16        |128            |16     |1024     |437.586                 |264.223                  |1.656                         |0.518  |
|torch.float16|16        |128            |16     |1024     |425.574                 |264.346                  |1.610                         |0.549  |
|torch.float16|16        |128            |16     |1024     |420.475                 |256.625                  |1.638                         |0.579  |
|torch.float16|16        |128            |16     |1024     |421.345                 |254.833                  |1.653                         |0.603  |
|torch.float16|16        |128            |16     |1024     |419.912                 |253.000                  |1.660                         |0.626  |
|torch.float16|16        |128            |16     |1024     |419.348                 |255.212                  |1.643                         |0.646  |
|torch.float16|16        |128            |16     |1024     |424.161                 |262.257                  |1.617                         |0.661  |
|torch.float16|16        |128            |16     |1024     |428.431                 |260.106                  |1.647                         |0.679  |
|torch.float16|16        |128            |16     |1024     |427.459                 |264.397                  |1.617                         |0.690  |
|torch.float16|16        |128            |16     |1024     |428.237                 |261.376                  |1.638                         |0.706  |
|torch.float16|16        |256            |8      |512      |398.295                 |262.605                  |1.517                         |0.077  |
|torch.float16|16        |256            |8      |512      |398.326                 |253.000                  |1.574                         |0.242  |
|torch.float16|16        |256            |8      |512      |399.821                 |253.819                  |1.575                         |0.372  |
|torch.float16|16        |256            |8      |512      |395.305                 |256.707                  |1.540                         |0.457  |
|torch.float16|16        |256            |8      |512      |407.798                 |255.068                  |1.599                         |0.507  |
|torch.float16|16        |256            |8      |512      |409.324                 |266.404                  |1.536                         |0.552  |
|torch.float16|16        |256            |8      |512      |398.787                 |255.836                  |1.559                         |0.517  |
|torch.float16|16        |256            |8      |512      |399.534                 |260.854                  |1.532                         |0.548  |
|torch.float16|16        |256            |8      |512      |396.503                 |253.020                  |1.567                         |0.578  |
|torch.float16|16        |256            |8      |512      |400.558                 |257.874                  |1.553                         |0.603  |
|torch.float16|16        |256            |8      |512      |405.453                 |257.638                  |1.574                         |0.624  |
|torch.float16|16        |256            |8      |512      |404.163                 |258.570                  |1.563                         |0.643  |
|torch.float16|16        |256            |8      |512      |393.339                 |254.618                  |1.545                         |0.662  |
|torch.float16|16        |256            |8      |512      |401.961                 |264.796                  |1.518                         |0.676  |
|torch.float16|16        |256            |8      |512      |402.350                 |258.150                  |1.559                         |0.690  |
|torch.float16|16        |256            |8      |512      |400.271                 |254.403                  |1.573                         |0.703  |
|torch.float16|16        |256            |16     |512      |398.305                 |251.290                  |1.585                         |0.077  |
|torch.float16|16        |256            |16     |512      |403.794                 |257.137                  |1.570                         |0.242  |
|torch.float16|16        |256            |16     |512      |400.855                 |261.786                  |1.531                         |0.372  |
|torch.float16|16        |256            |16     |512      |403.446                 |294.697                  |1.369                         |0.457  |
|torch.float16|16        |256            |16     |512      |406.979                 |268.124                  |1.518                         |0.507  |
|torch.float16|16        |256            |16     |512      |400.353                 |262.482                  |1.525                         |0.552  |
|torch.float16|16        |256            |16     |512      |395.366                 |260.721                  |1.516                         |0.517  |
|torch.float16|16        |256            |16     |512      |395.571                 |256.276                  |1.544                         |0.548  |
|torch.float16|16        |256            |16     |512      |400.343                 |254.956                  |1.570                         |0.578  |
|torch.float16|16        |256            |16     |512      |406.405                 |252.201                  |1.611                         |0.603  |
|torch.float16|16        |256            |16     |512      |396.431                 |257.004                  |1.543                         |0.624  |
|torch.float16|16        |256            |16     |512      |402.289                 |260.352                  |1.545                         |0.643  |
|torch.float16|16        |256            |16     |512      |406.139                 |255.908                  |1.587                         |0.662  |
|torch.float16|16        |256            |16     |512      |395.192                 |251.607                  |1.571                         |0.676  |
|torch.float16|16        |256            |16     |512      |397.015                 |258.079                  |1.538                         |0.690  |
|torch.float16|16        |256            |16     |512      |400.077                 |254.781                  |1.570                         |0.703  |
|torch.float16|16        |256            |8      |1024     |477.420                 |302.510                  |1.578                         |0.077  |
|torch.float16|16        |256            |8      |1024     |479.601                 |282.829                  |1.696                         |0.242  |
|torch.float16|16        |256            |8      |1024     |482.028                 |269.988                  |1.785                         |0.372  |
|torch.float16|16        |256            |8      |1024     |488.643                 |273.889                  |1.784                         |0.457  |
|torch.float16|16        |256            |8      |1024     |482.294                 |263.373                  |1.831                         |0.507  |
|torch.float16|16        |256            |8      |1024     |483.523                 |272.364                  |1.775                         |0.552  |
|torch.float16|16        |256            |8      |1024     |482.314                 |266.742                  |1.808                         |0.517  |
|torch.float16|16        |256            |8      |1024     |482.796                 |273.900                  |1.763                         |0.548  |
|torch.float16|16        |256            |8      |1024     |484.669                 |276.664                  |1.752                         |0.578  |
|torch.float16|16        |256            |8      |1024     |484.024                 |264.673                  |1.829                         |0.603  |
|torch.float16|16        |256            |8      |1024     |484.413                 |265.492                  |1.825                         |0.624  |
|torch.float16|16        |256            |8      |1024     |483.430                 |260.721                  |1.854                         |0.643  |
|torch.float16|16        |256            |8      |1024     |485.048                 |272.271                  |1.781                         |0.662  |
|torch.float16|16        |256            |8      |1024     |484.434                 |273.930                  |1.768                         |0.676  |
|torch.float16|16        |256            |8      |1024     |484.792                 |261.796                  |1.852                         |0.690  |
|torch.float16|16        |256            |8      |1024     |484.383                 |267.868                  |1.808                         |0.703  |
|torch.float16|16        |256            |16     |1024     |594.033                 |309.258                  |1.921                         |0.077  |
|torch.float16|16        |256            |16     |1024     |598.170                 |286.904                  |2.085                         |0.242  |
|torch.float16|16        |256            |16     |1024     |600.730                 |275.548                  |2.180                         |0.372  |
|torch.float16|16        |256            |16     |1024     |599.388                 |270.694                  |2.214                         |0.457  |
|torch.float16|16        |256            |16     |1024     |600.596                 |273.582                  |2.195                         |0.507  |
|torch.float16|16        |256            |16     |1024     |602.696                 |274.135                  |2.199                         |0.552  |
|torch.float16|16        |256            |16     |1024     |601.498                 |267.745                  |2.247                         |0.517  |
|torch.float16|16        |256            |16     |1024     |602.235                 |273.633                  |2.201                         |0.548  |
|torch.float16|16        |256            |16     |1024     |602.317                 |271.247                  |2.221                         |0.578  |
|torch.float16|16        |256            |16     |1024     |602.972                 |260.997                  |2.310                         |0.603  |
|torch.float16|16        |256            |16     |1024     |602.900                 |265.513                  |2.271                         |0.624  |
|torch.float16|16        |256            |16     |1024     |603.197                 |262.758                  |2.296                         |0.643  |
|torch.float16|16        |256            |16     |1024     |604.170                 |273.254                  |2.211                         |0.662  |
|torch.float16|16        |256            |16     |1024     |603.208                 |267.930                  |2.251                         |0.676  |
|torch.float16|16        |256            |16     |1024     |603.423                 |266.076                  |2.268                         |0.690  |
|torch.float16|16        |256            |16     |1024     |603.689                 |267.981                  |2.253                         |0.703  |
|torch.float16|16        |512            |8      |512      |726.487                 |269.834                  |2.692                         |0.077  |
|torch.float16|16        |512            |8      |512      |734.382                 |260.608                  |2.818                         |0.242  |
|torch.float16|16        |512            |8      |512      |741.550                 |263.219                  |2.817                         |0.370  |
|torch.float16|16        |512            |8      |512      |741.315                 |262.728                  |2.822                         |0.456  |
|torch.float16|16        |512            |8      |512      |741.540                 |259.021                  |2.863                         |0.507  |
|torch.float16|16        |512            |8      |512      |745.247                 |259.205                  |2.875                         |0.552  |
|torch.float16|16        |512            |8      |512      |744.192                 |265.329                  |2.805                         |0.515  |
|torch.float16|16        |512            |8      |512      |745.769                 |262.513                  |2.841                         |0.548  |
|torch.float16|16        |512            |8      |512      |746.988                 |260.608                  |2.866                         |0.577  |
|torch.float16|16        |512            |8      |512      |747.960                 |264.069                  |2.832                         |0.601  |
|torch.float16|16        |512            |8      |512      |748.083                 |261.427                  |2.862                         |0.623  |
|torch.float16|16        |512            |8      |512      |749.097                 |263.526                  |2.843                         |0.642  |
|torch.float16|16        |512            |8      |512      |750.346                 |269.302                  |2.786                         |0.660  |
|torch.float16|16        |512            |8      |512      |750.172                 |259.748                  |2.888                         |0.676  |
|torch.float16|16        |512            |8      |512      |750.039                 |261.806                  |2.865                         |0.689  |
|torch.float16|16        |512            |8      |512      |750.244                 |259.594                  |2.890                         |0.703  |
|torch.float16|16        |512            |16     |512      |1119.642                |322.693                  |3.470                         |0.077  |
|torch.float16|16        |512            |16     |512      |1130.701                |282.573                  |4.001                         |0.242  |
|torch.float16|16        |512            |16     |512      |1141.699                |260.219                  |4.387                         |0.370  |
|torch.float16|16        |512            |16     |512      |1146.122                |267.786                  |4.280                         |0.456  |
|torch.float16|16        |512            |16     |512      |1147.627                |265.380                  |4.324                         |0.507  |
|torch.float16|16        |512            |16     |512      |1154.468                |266.383                  |4.334                         |0.552  |
|torch.float16|16        |512            |16     |512      |1148.795                |262.339                  |4.379                         |0.515  |
|torch.float16|16        |512            |16     |512      |1150.894                |261.151                  |4.407                         |0.548  |
|torch.float16|16        |512            |16     |512      |1153.792                |265.216                  |4.350                         |0.577  |
|torch.float16|16        |512            |16     |512      |1154.427                |263.270                  |4.385                         |0.601  |
|torch.float16|16        |512            |16     |512      |1157.141                |262.523                  |4.408                         |0.623  |
|torch.float16|16        |512            |16     |512      |1158.236                |259.492                  |4.463                         |0.642  |
|torch.float16|16        |512            |16     |512      |1160.458                |260.649                  |4.452                         |0.660  |
|torch.float16|16        |512            |16     |512      |1162.373                |264.151                  |4.400                         |0.676  |
|torch.float16|16        |512            |16     |512      |1162.250                |260.567                  |4.460                         |0.689  |
|torch.float16|16        |512            |16     |512      |1163.735                |257.638                  |4.517                         |0.703  |
|torch.float16|16        |512            |8      |1024     |1143.306                |507.750                  |2.252                         |0.077  |
|torch.float16|16        |512            |8      |1024     |1146.378                |429.701                  |2.668                         |0.242  |
|torch.float16|16        |512            |8      |1024     |1149.348                |375.818                  |3.058                         |0.370  |
|torch.float16|16        |512            |8      |1024     |1149.245                |324.762                  |3.539                         |0.456  |
|torch.float16|16        |512            |8      |1024     |1151.918                |341.279                  |3.375                         |0.507  |
|torch.float16|16        |512            |8      |1024     |1154.447                |330.148                  |3.497                         |0.552  |
|torch.float16|16        |512            |8      |1024     |1153.198                |339.558                  |3.396                         |0.515  |
|torch.float16|16        |512            |8      |1024     |1154.191                |322.580                  |3.578                         |0.548  |
|torch.float16|16        |512            |8      |1024     |1155.226                |320.102                  |3.609                         |0.577  |
|torch.float16|16        |512            |8      |1024     |1154.908                |283.535                  |4.073                         |0.601  |
|torch.float16|16        |512            |8      |1024     |1155.379                |282.982                  |4.083                         |0.623  |
|torch.float16|16        |512            |8      |1024     |1159.219                |294.615                  |3.935                         |0.642  |
|torch.float16|16        |512            |8      |1024     |1159.434                |277.422                  |4.179                         |0.660  |
|torch.float16|16        |512            |8      |1024     |1159.885                |271.012                  |4.280                         |0.676  |
|torch.float16|16        |512            |8      |1024     |1158.656                |287.293                  |4.033                         |0.689  |
|torch.float16|16        |512            |8      |1024     |1159.178                |284.324                  |4.077                         |0.703  |
|torch.float16|16        |512            |16     |1024     |1533.389                |543.980                  |2.819                         |0.077  |
|torch.float16|16        |512            |16     |1024     |1544.120                |462.469                  |3.339                         |0.242  |
|torch.float16|16        |512            |16     |1024     |1550.715                |398.285                  |3.893                         |0.370  |
|torch.float16|16        |512            |16     |1024     |1554.985                |339.241                  |4.584                         |0.456  |
|torch.float16|16        |512            |16     |1024     |1557.811                |352.717                  |4.417                         |0.507  |
|torch.float16|16        |512            |16     |1024     |1564.201                |325.786                  |4.801                         |0.552  |
|torch.float16|16        |512            |16     |1024     |1559.204                |347.587                  |4.486                         |0.515  |
|torch.float16|16        |512            |16     |1024     |1560.842                |328.489                  |4.752                         |0.548  |
|torch.float16|16        |512            |16     |1024     |1563.505                |319.468                  |4.894                         |0.577  |
|torch.float16|16        |512            |16     |1024     |1563.679                |284.887                  |5.489                         |0.601  |
|torch.float16|16        |512            |16     |1024     |1566.198                |282.696                  |5.540                         |0.623  |
|torch.float16|16        |512            |16     |1024     |1569.270                |298.353                  |5.260                         |0.642  |
|torch.float16|16        |512            |16     |1024     |1570.816                |272.005                  |5.775                         |0.660  |
|torch.float16|16        |512            |16     |1024     |1572.915                |273.050                  |5.761                         |0.676  |
|torch.float16|16        |512            |16     |1024     |1572.311                |283.904                  |5.538                         |0.689  |
|torch.float16|16        |512            |16     |1024     |1573.642                |282.614                  |5.568                         |0.703  |
|torch.float16|32        |64             |8      |512      |414.720                 |286.822                  |1.446                         |0.050  |
|torch.float16|32        |64             |8      |512      |415.672                 |277.350                  |1.499                         |0.223  |
|torch.float16|32        |64             |8      |512      |439.163                 |278.651                  |1.576                         |0.332  |
|torch.float16|32        |64             |8      |512      |408.924                 |281.078                  |1.455                         |0.401  |
|torch.float16|32        |64             |8      |512      |407.429                 |283.013                  |1.440                         |0.476  |
|torch.float16|32        |64             |8      |512      |414.730                 |282.214                  |1.470                         |0.504  |
|torch.float16|32        |64             |8      |512      |419.359                 |291.594                  |1.438                         |0.514  |
|torch.float16|32        |64             |8      |512      |406.620                 |287.181                  |1.416                         |0.551  |
|torch.float16|32        |64             |8      |512      |410.235                 |282.941                  |1.450                         |0.573  |
|torch.float16|32        |64             |8      |512      |409.754                 |280.873                  |1.459                         |0.598  |
|torch.float16|32        |64             |8      |512      |409.836                 |286.689                  |1.430                         |0.623  |
|torch.float16|32        |64             |8      |512      |405.524                 |280.771                  |1.444                         |0.644  |
|torch.float16|32        |64             |8      |512      |408.269                 |279.183                  |1.462                         |0.663  |
|torch.float16|32        |64             |8      |512      |413.747                 |275.773                  |1.500                         |0.681  |
|torch.float16|32        |64             |8      |512      |406.569                 |277.248                  |1.466                         |0.695  |
|torch.float16|32        |64             |8      |512      |408.852                 |285.338                  |1.433                         |0.706  |
|torch.float16|32        |64             |16     |512      |412.006                 |285.952                  |1.441                         |0.050  |
|torch.float16|32        |64             |16     |512      |409.426                 |279.562                  |1.465                         |0.223  |
|torch.float16|32        |64             |16     |512      |408.586                 |277.934                  |1.470                         |0.332  |
|torch.float16|32        |64             |16     |512      |404.716                 |282.173                  |1.434                         |0.401  |
|torch.float16|32        |64             |16     |512      |421.130                 |279.859                  |1.505                         |0.476  |
|torch.float16|32        |64             |16     |512      |413.481                 |287.048                  |1.440                         |0.504  |
|torch.float16|32        |64             |16     |512      |409.641                 |321.413                  |1.274                         |0.514  |
|torch.float16|32        |64             |16     |512      |406.927                 |283.689                  |1.434                         |0.551  |
|torch.float16|32        |64             |16     |512      |409.088                 |279.009                  |1.466                         |0.573  |
|torch.float16|32        |64             |16     |512      |414.566                 |283.699                  |1.461                         |0.598  |
|torch.float16|32        |64             |16     |512      |405.514                 |283.474                  |1.431                         |0.623  |
|torch.float16|32        |64             |16     |512      |406.886                 |286.085                  |1.422                         |0.644  |
|torch.float16|32        |64             |16     |512      |415.908                 |280.207                  |1.484                         |0.663  |
|torch.float16|32        |64             |16     |512      |409.651                 |277.504                  |1.476                         |0.681  |
|torch.float16|32        |64             |16     |512      |406.917                 |282.778                  |1.439                         |0.695  |
|torch.float16|32        |64             |16     |512      |404.050                 |286.351                  |1.411                         |0.706  |
|torch.float16|32        |64             |8      |1024     |422.144                 |294.236                  |1.435                         |0.050  |
|torch.float16|32        |64             |8      |1024     |425.165                 |282.286                  |1.506                         |0.223  |
|torch.float16|32        |64             |8      |1024     |426.629                 |289.157                  |1.475                         |0.332  |
|torch.float16|32        |64             |8      |1024     |425.073                 |294.031                  |1.446                         |0.401  |
|torch.float16|32        |64             |8      |1024     |420.280                 |286.761                  |1.466                         |0.476  |
|torch.float16|32        |64             |8      |1024     |421.448                 |281.119                  |1.499                         |0.504  |
|torch.float16|32        |64             |8      |1024     |422.574                 |283.136                  |1.492                         |0.514  |
|torch.float16|32        |64             |8      |1024     |421.550                 |289.280                  |1.457                         |0.551  |
|torch.float16|32        |64             |8      |1024     |424.591                 |283.771                  |1.496                         |0.573  |
|torch.float16|32        |64             |8      |1024     |434.565                 |280.689                  |1.548                         |0.598  |
|torch.float16|32        |64             |8      |1024     |416.881                 |287.795                  |1.449                         |0.623  |
|torch.float16|32        |64             |8      |1024     |423.926                 |288.072                  |1.472                         |0.644  |
|torch.float16|32        |64             |8      |1024     |418.570                 |284.396                  |1.472                         |0.663  |
|torch.float16|32        |64             |8      |1024     |423.895                 |283.116                  |1.497                         |0.681  |
|torch.float16|32        |64             |8      |1024     |419.246                 |279.060                  |1.502                         |0.695  |
|torch.float16|32        |64             |8      |1024     |421.632                 |284.836                  |1.480                         |0.706  |
|torch.float16|32        |64             |16     |1024     |413.757                 |284.979                  |1.452                         |0.050  |
|torch.float16|32        |64             |16     |1024     |413.542                 |289.116                  |1.430                         |0.223  |
|torch.float16|32        |64             |16     |1024     |407.460                 |289.556                  |1.407                         |0.332  |
|torch.float16|32        |64             |16     |1024     |408.566                 |288.276                  |1.417                         |0.401  |
|torch.float16|32        |64             |16     |1024     |412.129                 |282.624                  |1.458                         |0.476  |
|torch.float16|32        |64             |16     |1024     |409.211                 |283.238                  |1.445                         |0.504  |
|torch.float16|32        |64             |16     |1024     |406.764                 |284.467                  |1.430                         |0.514  |
|torch.float16|32        |64             |16     |1024     |409.805                 |288.461                  |1.421                         |0.551  |
|torch.float16|32        |64             |16     |1024     |415.734                 |281.068                  |1.479                         |0.573  |
|torch.float16|32        |64             |16     |1024     |410.552                 |283.361                  |1.449                         |0.598  |
|torch.float16|32        |64             |16     |1024     |407.706                 |280.596                  |1.453                         |0.623  |
|torch.float16|32        |64             |16     |1024     |406.927                 |284.334                  |1.431                         |0.644  |
|torch.float16|32        |64             |16     |1024     |416.369                 |279.091                  |1.492                         |0.663  |
|torch.float16|32        |64             |16     |1024     |408.433                 |280.699                  |1.455                         |0.681  |
|torch.float16|32        |64             |16     |1024     |410.132                 |278.774                  |1.471                         |0.695  |
|torch.float16|32        |64             |16     |1024     |417.413                 |289.608                  |1.441                         |0.706  |
|torch.float16|32        |128            |8      |512      |430.725                 |280.914                  |1.533                         |0.049  |
|torch.float16|32        |128            |8      |512      |403.446                 |279.747                  |1.442                         |0.225  |
|torch.float16|32        |128            |8      |512      |400.763                 |282.020                  |1.421                         |0.332  |
|torch.float16|32        |128            |8      |512      |405.576                 |280.965                  |1.444                         |0.398  |
|torch.float16|32        |128            |8      |512      |403.282                 |276.910                  |1.456                         |0.473  |
|torch.float16|32        |128            |8      |512      |400.302                 |288.031                  |1.390                         |0.501  |
|torch.float16|32        |128            |8      |512      |399.380                 |286.239                  |1.395                         |0.512  |
|torch.float16|32        |128            |8      |512      |397.640                 |281.866                  |1.411                         |0.544  |
|torch.float16|32        |128            |8      |512      |403.190                 |280.320                  |1.438                         |0.569  |
|torch.float16|32        |128            |8      |512      |399.094                 |281.027                  |1.420                         |0.595  |
|torch.float16|32        |128            |8      |512      |402.022                 |282.962                  |1.421                         |0.620  |
|torch.float16|32        |128            |8      |512      |399.493                 |274.350                  |1.456                         |0.641  |
|torch.float16|32        |128            |8      |512      |399.186                 |280.197                  |1.425                         |0.660  |
|torch.float16|32        |128            |8      |512      |406.876                 |278.764                  |1.460                         |0.676  |
|torch.float16|32        |128            |8      |512      |406.016                 |290.540                  |1.397                         |0.691  |
|torch.float16|32        |128            |8      |512      |404.296                 |280.934                  |1.439                         |0.707  |
|torch.float16|32        |128            |16     |512      |398.746                 |279.009                  |1.429                         |0.049  |
|torch.float16|32        |128            |16     |512      |399.862                 |284.662                  |1.405                         |0.225  |
|torch.float16|32        |128            |16     |512      |408.136                 |280.965                  |1.453                         |0.332  |
|torch.float16|32        |128            |16     |512      |405.647                 |280.893                  |1.444                         |0.398  |
|torch.float16|32        |128            |16     |512      |398.244                 |276.695                  |1.439                         |0.473  |
|torch.float16|32        |128            |16     |512      |398.193                 |283.976                  |1.402                         |0.501  |
|torch.float16|32        |128            |16     |512      |402.729                 |281.047                  |1.433                         |0.512  |
|torch.float16|32        |128            |16     |512      |404.091                 |279.368                  |1.446                         |0.544  |
|torch.float16|32        |128            |16     |512      |421.734                 |278.436                  |1.515                         |0.569  |
|torch.float16|32        |128            |16     |512      |396.995                 |281.242                  |1.412                         |0.595  |
|torch.float16|32        |128            |16     |512      |402.412                 |281.559                  |1.429                         |0.620  |
|torch.float16|32        |128            |16     |512      |399.759                 |279.849                  |1.428                         |0.641  |
|torch.float16|32        |128            |16     |512      |408.699                 |283.679                  |1.441                         |0.660  |
|torch.float16|32        |128            |16     |512      |400.435                 |283.167                  |1.414                         |0.676  |
|torch.float16|32        |128            |16     |512      |405.821                 |280.197                  |1.448                         |0.691  |
|torch.float16|32        |128            |16     |512      |396.769                 |279.368                  |1.420                         |0.707  |
|torch.float16|32        |128            |8      |1024     |428.984                 |321.710                  |1.333                         |0.049  |
|torch.float16|32        |128            |8      |1024     |432.169                 |298.680                  |1.447                         |0.225  |
|torch.float16|32        |128            |8      |1024     |431.391                 |285.542                  |1.511                         |0.332  |
|torch.float16|32        |128            |8      |1024     |430.541                 |297.544                  |1.447                         |0.398  |
|torch.float16|32        |128            |8      |1024     |432.251                 |286.648                  |1.508                         |0.473  |
|torch.float16|32        |128            |8      |1024     |433.469                 |282.522                  |1.534                         |0.501  |
|torch.float16|32        |128            |8      |1024     |435.036                 |283.750                  |1.533                         |0.512  |
|torch.float16|32        |128            |8      |1024     |432.343                 |285.870                  |1.512                         |0.544  |
|torch.float16|32        |128            |8      |1024     |432.783                 |286.208                  |1.512                         |0.569  |
|torch.float16|32        |128            |8      |1024     |433.428                 |280.873                  |1.543                         |0.595  |
|torch.float16|32        |128            |8      |1024     |436.972                 |287.293                  |1.521                         |0.620  |
|torch.float16|32        |128            |8      |1024     |433.766                 |283.290                  |1.531                         |0.641  |
|torch.float16|32        |128            |8      |1024     |434.596                 |288.399                  |1.507                         |0.660  |
|torch.float16|32        |128            |8      |1024     |432.599                 |293.489                  |1.474                         |0.676  |
|torch.float16|32        |128            |8      |1024     |434.452                 |291.922                  |1.488                         |0.691  |
|torch.float16|32        |128            |8      |1024     |434.360                 |310.804                  |1.398                         |0.707  |
|torch.float16|32        |128            |16     |1024     |476.815                 |319.293                  |1.493                         |0.049  |
|torch.float16|32        |128            |16     |1024     |479.406                 |293.847                  |1.631                         |0.225  |
|torch.float16|32        |128            |16     |1024     |481.413                 |285.153                  |1.688                         |0.332  |
|torch.float16|32        |128            |16     |1024     |481.260                 |301.568                  |1.596                         |0.398  |
|torch.float16|32        |128            |16     |1024     |482.335                 |284.508                  |1.695                         |0.473  |
|torch.float16|32        |128            |16     |1024     |483.748                 |280.463                  |1.725                         |0.501  |
|torch.float16|32        |128            |16     |1024     |483.226                 |282.481                  |1.711                         |0.512  |
|torch.float16|32        |128            |16     |1024     |484.116                 |292.588                  |1.655                         |0.544  |
|torch.float16|32        |128            |16     |1024     |483.748                 |292.567                  |1.653                         |0.569  |
|torch.float16|32        |128            |16     |1024     |484.311                 |284.406                  |1.703                         |0.595  |
|torch.float16|32        |128            |16     |1024     |483.871                 |279.470                  |1.731                         |0.620  |
|torch.float16|32        |128            |16     |1024     |484.659                 |277.432                  |1.747                         |0.641  |
|torch.float16|32        |128            |16     |1024     |484.065                 |285.092                  |1.698                         |0.660  |
|torch.float16|32        |128            |16     |1024     |485.243                 |285.522                  |1.699                         |0.676  |
|torch.float16|32        |128            |16     |1024     |484.741                 |289.894                  |1.672                         |0.691  |
|torch.float16|32        |128            |16     |1024     |484.352                 |288.410                  |1.679                         |0.707  |
|torch.float16|32        |256            |8      |512      |525.384                 |280.996                  |1.870                         |0.049  |
|torch.float16|32        |256            |8      |512      |526.019                 |277.043                  |1.899                         |0.223  |
|torch.float16|32        |256            |8      |512      |527.155                 |279.040                  |1.889                         |0.331  |
|torch.float16|32        |256            |8      |512      |529.644                 |280.033                  |1.891                         |0.398  |
|torch.float16|32        |256            |8      |512      |531.016                 |313.989                  |1.691                         |0.472  |
|torch.float16|32        |256            |8      |512      |532.777                 |287.549                  |1.853                         |0.500  |
|torch.float16|32        |256            |8      |512      |529.940                 |280.945                  |1.886                         |0.510  |
|torch.float16|32        |256            |8      |512      |531.016                 |277.012                  |1.917                         |0.544  |
|torch.float16|32        |256            |8      |512      |530.842                 |277.688                  |1.912                         |0.567  |
|torch.float16|32        |256            |8      |512      |532.111                 |278.395                  |1.911                         |0.595  |
|torch.float16|32        |256            |8      |512      |532.081                 |276.695                  |1.923                         |0.617  |
|torch.float16|32        |256            |8      |512      |532.511                 |274.002                  |1.943                         |0.638  |
|torch.float16|32        |256            |8      |512      |534.200                 |280.484                  |1.905                         |0.658  |
|torch.float16|32        |256            |8      |512      |534.999                 |279.982                  |1.911                         |0.674  |
|torch.float16|32        |256            |8      |512      |534.067                 |279.706                  |1.909                         |0.689  |
|torch.float16|32        |256            |8      |512      |535.173                 |275.313                  |1.944                         |0.703  |
|torch.float16|32        |256            |16     |512      |724.859                 |284.600                  |2.547                         |0.049  |
|torch.float16|32        |256            |16     |512      |730.655                 |290.181                  |2.518                         |0.223  |
|torch.float16|32        |256            |16     |512      |734.525                 |286.556                  |2.563                         |0.331  |
|torch.float16|32        |256            |16     |512      |735.386                 |291.574                  |2.522                         |0.398  |
|torch.float16|32        |256            |16     |512      |736.533                 |291.256                  |2.529                         |0.472  |
|torch.float16|32        |256            |16     |512      |736.819                 |285.409                  |2.582                         |0.500  |
|torch.float16|32        |256            |16     |512      |739.420                 |282.102                  |2.621                         |0.510  |
|torch.float16|32        |256            |16     |512      |739.891                 |282.614                  |2.618                         |0.544  |
|torch.float16|32        |256            |16     |512      |740.608                 |295.885                  |2.503                         |0.567  |
|torch.float16|32        |256            |16     |512      |741.478                 |317.594                  |2.335                         |0.595  |
|torch.float16|32        |256            |16     |512      |742.185                 |293.130                  |2.532                         |0.617  |
|torch.float16|32        |256            |16     |512      |742.154                 |287.017                  |2.586                         |0.638  |
|torch.float16|32        |256            |16     |512      |745.175                 |278.016                  |2.680                         |0.658  |
|torch.float16|32        |256            |16     |512      |747.090                 |280.924                  |2.659                         |0.674  |
|torch.float16|32        |256            |16     |512      |746.373                 |283.423                  |2.633                         |0.689  |
|torch.float16|32        |256            |16     |512      |746.670                 |277.893                  |2.687                         |0.703  |
|torch.float16|32        |256            |8      |1024     |946.534                 |447.826                  |2.114                         |0.049  |
|torch.float16|32        |256            |8      |1024     |939.981                 |393.677                  |2.388                         |0.223  |
|torch.float16|32        |256            |8      |1024     |941.046                 |359.752                  |2.616                         |0.331  |
|torch.float16|32        |256            |8      |1024     |943.503                 |350.382                  |2.693                         |0.398  |
|torch.float16|32        |256            |8      |1024     |944.210                 |324.905                  |2.906                         |0.472  |
|torch.float16|32        |256            |8      |1024     |946.207                 |349.071                  |2.711                         |0.500  |
|torch.float16|32        |256            |8      |1024     |944.701                 |349.809                  |2.701                         |0.510  |
|torch.float16|32        |256            |8      |1024     |945.357                 |317.573                  |2.977                         |0.544  |
|torch.float16|32        |256            |8      |1024     |945.818                 |333.496                  |2.836                         |0.567  |
|torch.float16|32        |256            |8      |1024     |945.797                 |294.359                  |3.213                         |0.595  |
|torch.float16|32        |256            |8      |1024     |945.879                 |298.209                  |3.172                         |0.617  |
|torch.float16|32        |256            |8      |1024     |946.278                 |315.894                  |2.996                         |0.638  |
|torch.float16|32        |256            |8      |1024     |948.521                 |285.512                  |3.322                         |0.658  |
|torch.float16|32        |256            |8      |1024     |949.637                 |287.672                  |3.301                         |0.674  |
|torch.float16|32        |256            |8      |1024     |948.050                 |302.049                  |3.139                         |0.689  |
|torch.float16|32        |256            |8      |1024     |948.695                 |297.308                  |3.191                         |0.703  |
|torch.float16|32        |256            |16     |1024     |1136.824                |463.309                  |2.454                         |0.049  |
|torch.float16|32        |256            |16     |1024     |1136.927                |403.261                  |2.819                         |0.223  |
|torch.float16|32        |256            |16     |1024     |1139.528                |366.848                  |3.106                         |0.331  |
|torch.float16|32        |256            |16     |1024     |1140.982                |346.030                  |3.297                         |0.398  |
|torch.float16|32        |256            |16     |1024     |1141.453                |323.963                  |3.523                         |0.472  |
|torch.float16|32        |256            |16     |1024     |1142.467                |353.167                  |3.235                         |0.500  |
|torch.float16|32        |256            |16     |1024     |1145.498                |350.915                  |3.264                         |0.510  |
|torch.float16|32        |256            |16     |1024     |1145.651                |316.590                  |3.619                         |0.544  |
|torch.float16|32        |256            |16     |1024     |1145.856                |330.977                  |3.462                         |0.567  |
|torch.float16|32        |256            |16     |1024     |1146.890                |303.094                  |3.784                         |0.595  |
|torch.float16|32        |256            |16     |1024     |1147.607                |293.888                  |3.905                         |0.617  |
|torch.float16|32        |256            |16     |1024     |1147.679                |308.756                  |3.717                         |0.638  |
|torch.float16|32        |256            |16     |1024     |1150.730                |284.375                  |4.047                         |0.658  |
|torch.float16|32        |256            |16     |1024     |1153.126                |289.137                  |3.988                         |0.674  |
|torch.float16|32        |256            |16     |1024     |1151.795                |303.155                  |3.799                         |0.689  |
|torch.float16|32        |256            |16     |1024     |1151.969                |296.387                  |3.887                         |0.703  |
|torch.float16|32        |512            |8      |512      |1391.237                |430.356                  |3.233                         |0.049  |
|torch.float16|32        |512            |8      |512      |1406.310                |363.756                  |3.866                         |0.223  |
|torch.float16|32        |512            |8      |512      |1414.380                |328.387                  |4.307                         |0.329  |
|torch.float16|32        |512            |8      |512      |1417.718                |300.913                  |4.711                         |0.397  |
|torch.float16|32        |512            |8      |512      |1421.107                |290.816                  |4.887                         |0.471  |
|torch.float16|32        |512            |8      |512      |1422.244                |287.478                  |4.947                         |0.499  |
|torch.float16|32        |512            |8      |512      |1424.988                |286.659                  |4.971                         |0.509  |
|torch.float16|32        |512            |8      |512      |1426.401                |283.443                  |5.032                         |0.543  |
|torch.float16|32        |512            |8      |512      |1428.326                |288.369                  |4.953                         |0.566  |
|torch.float16|32        |512            |8      |512      |1430.559                |286.761                  |4.989                         |0.593  |
|torch.float16|32        |512            |8      |512      |1431.951                |286.546                  |4.997                         |0.617  |
|torch.float16|32        |512            |8      |512      |1433.508                |283.689                  |5.053                         |0.638  |
|torch.float16|32        |512            |8      |512      |1435.638                |284.662                  |5.043                         |0.657  |
|torch.float16|32        |512            |8      |512      |1436.559                |283.638                  |5.065                         |0.673  |
|torch.float16|32        |512            |8      |512      |1437.317                |286.454                  |5.018                         |0.688  |
|torch.float16|32        |512            |8      |512      |1438.874                |287.683                  |5.002                         |0.702  |
|torch.float16|32        |512            |16     |512      |2145.393                |574.525                  |3.734                         |0.049  |
|torch.float16|32        |512            |16     |512      |2166.856                |488.346                  |4.437                         |0.223  |
|torch.float16|32        |512            |16     |512      |2181.745                |424.991                  |5.134                         |0.329  |
|torch.float16|32        |512            |16     |512      |2191.749                |368.599                  |5.946                         |0.397  |
|torch.float16|32        |512            |16     |512      |2201.160                |337.050                  |6.531                         |0.471  |
|torch.float16|32        |512            |16     |512      |2206.669                |320.707                  |6.881                         |0.499  |
|torch.float16|32        |512            |16     |512      |2206.679                |314.337                  |7.020                         |0.509  |
|torch.float16|32        |512            |16     |512      |2210.038                |300.759                  |7.348                         |0.543  |
|torch.float16|32        |512            |16     |512      |2213.356                |295.465                  |7.491                         |0.566  |
|torch.float16|32        |512            |16     |512      |2217.236                |290.949                  |7.621                         |0.593  |
|torch.float16|32        |512            |16     |512      |2223.278                |281.754                  |7.891                         |0.617  |
|torch.float16|32        |512            |16     |512      |2227.477                |286.761                  |7.768                         |0.638  |
|torch.float16|32        |512            |16     |512      |2230.712                |285.972                  |7.800                         |0.657  |
|torch.float16|32        |512            |16     |512      |2235.750                |288.686                  |7.745                         |0.673  |
|torch.float16|32        |512            |16     |512      |2237.215                |286.136                  |7.819                         |0.688  |
|torch.float16|32        |512            |16     |512      |2240.461                |281.508                  |7.959                         |0.702  |
|torch.float16|32        |512            |8      |1024     |2173.686                |993.608                  |2.188                         |0.049  |
|torch.float16|32        |512            |8      |1024     |2196.449                |816.097                  |2.691                         |0.223  |
|torch.float16|32        |512            |8      |1024     |2192.998                |733.993                  |2.988                         |0.329  |
|torch.float16|32        |512            |8      |1024     |2194.033                |635.423                  |3.453                         |0.397  |
|torch.float16|32        |512            |8      |1024     |2190.909                |558.756                  |3.921                         |0.471  |
|torch.float16|32        |512            |8      |1024     |2193.736                |525.814                  |4.172                         |0.499  |
|torch.float16|32        |512            |8      |1024     |2193.326                |511.406                  |4.289                         |0.509  |
|torch.float16|32        |512            |8      |1024     |2197.606                |479.672                  |4.581                         |0.543  |
|torch.float16|32        |512            |8      |1024     |2197.729                |456.264                  |4.817                         |0.566  |
|torch.float16|32        |512            |8      |1024     |2197.780                |413.256                  |5.318                         |0.593  |
|torch.float16|32        |512            |8      |1024     |2196.193                |414.587                  |5.297                         |0.617  |
|torch.float16|32        |512            |8      |1024     |2192.445                |393.503                  |5.572                         |0.638  |
|torch.float16|32        |512            |8      |1024     |2198.569                |372.009                  |5.910                         |0.657  |
|torch.float16|32        |512            |8      |1024     |2196.521                |367.299                  |5.980                         |0.673  |
|torch.float16|32        |512            |8      |1024     |2196.797                |353.864                  |6.208                         |0.688  |
|torch.float16|32        |512            |8      |1024     |2199.685                |341.668                  |6.438                         |0.702  |
|torch.float16|32        |512            |16     |1024     |2935.153                |1054.013                 |2.785                         |0.049  |
|torch.float16|32        |512            |16     |1024     |2956.165                |875.643                  |3.376                         |0.223  |
|torch.float16|32        |512            |16     |1024     |2967.849                |753.756                  |3.937                         |0.329  |
|torch.float16|32        |512            |16     |1024     |2975.252                |666.040                  |4.467                         |0.397  |
|torch.float16|32        |512            |16     |1024     |2982.840                |594.463                  |5.018                         |0.471  |
|torch.float16|32        |512            |16     |1024     |2985.564                |554.107                  |5.388                         |0.499  |
|torch.float16|32        |512            |16     |1024     |2985.032                |571.628                  |5.222                         |0.509  |
|torch.float16|32        |512            |16     |1024     |2987.448                |505.590                  |5.909                         |0.543  |
|torch.float16|32        |512            |16     |1024     |2991.749                |483.369                  |6.189                         |0.566  |
|torch.float16|32        |512            |16     |1024     |2994.668                |436.316                  |6.864                         |0.593  |
|torch.float16|32        |512            |16     |1024     |2997.863                |436.654                  |6.866                         |0.617  |
|torch.float16|32        |512            |16     |1024     |2999.736                |405.135                  |7.404                         |0.638  |
|torch.float16|32        |512            |16     |1024     |3003.515                |383.969                  |7.822                         |0.657  |
|torch.float16|32        |512            |16     |1024     |3008.481                |375.777                  |8.006                         |0.673  |
|torch.float16|32        |512            |16     |1024     |3009.557                |358.451                  |8.396                         |0.688  |
|torch.float16|32        |512            |16     |1024     |3013.694                |347.361                  |8.676                         |0.702  |
|torch.float16|64        |64             |8      |512      |390.267                 |338.125                  |1.154                         |0.034  |
|torch.float16|64        |64             |8      |512      |391.619                 |332.626                  |1.177                         |0.207  |
|torch.float16|64        |64             |8      |512      |386.181                 |332.411                  |1.162                         |0.337  |
|torch.float16|64        |64             |8      |512      |390.461                 |329.554                  |1.185                         |0.418  |
|torch.float16|64        |64             |8      |512      |389.683                 |331.776                  |1.175                         |0.479  |
|torch.float16|64        |64             |8      |512      |395.131                 |335.688                  |1.177                         |0.496  |
|torch.float16|64        |64             |8      |512      |386.806                 |342.979                  |1.128                         |0.524  |
|torch.float16|64        |64             |8      |512      |387.523                 |330.957                  |1.171                         |0.568  |
|torch.float16|64        |64             |8      |512      |388.659                 |330.220                  |1.177                         |0.583  |
|torch.float16|64        |64             |8      |512      |388.035                 |330.783                  |1.173                         |0.613  |
|torch.float16|64        |64             |8      |512      |391.547                 |335.340                  |1.168                         |0.640  |
|torch.float16|64        |64             |8      |512      |390.154                 |334.387                  |1.167                         |0.663  |
|torch.float16|64        |64             |8      |512      |421.274                 |334.551                  |1.259                         |0.683  |
|torch.float16|64        |64             |8      |512      |389.581                 |334.060                  |1.166                         |0.702  |
|torch.float16|64        |64             |8      |512      |387.666                 |334.326                  |1.160                         |0.716  |
|torch.float16|64        |64             |8      |512      |388.219                 |331.909                  |1.170                         |0.729  |
|torch.float16|64        |64             |16     |512      |385.966                 |330.424                  |1.168                         |0.034  |
|torch.float16|64        |64             |16     |512      |396.032                 |331.377                  |1.195                         |0.207  |
|torch.float16|64        |64             |16     |512      |389.468                 |331.172                  |1.176                         |0.337  |
|torch.float16|64        |64             |16     |512      |398.746                 |338.749                  |1.177                         |0.418  |
|torch.float16|64        |64             |16     |512      |393.124                 |336.169                  |1.169                         |0.479  |
|torch.float16|64        |64             |16     |512      |389.366                 |335.626                  |1.160                         |0.496  |
|torch.float16|64        |64             |16     |512      |391.649                 |337.510                  |1.160                         |0.524  |
|torch.float16|64        |64             |16     |512      |389.448                 |332.411                  |1.172                         |0.568  |
|torch.float16|64        |64             |16     |512      |392.550                 |333.414                  |1.177                         |0.583  |
|torch.float16|64        |64             |16     |512      |397.496                 |332.718                  |1.195                         |0.613  |
|torch.float16|64        |64             |16     |512      |392.970                 |337.613                  |1.164                         |0.640  |
|torch.float16|64        |64             |16     |512      |391.885                 |336.497                  |1.165                         |0.663  |
|torch.float16|64        |64             |16     |512      |389.140                 |330.578                  |1.177                         |0.683  |
|torch.float16|64        |64             |16     |512      |387.287                 |333.558                  |1.161                         |0.702  |
|torch.float16|64        |64             |16     |512      |392.417                 |329.359                  |1.191                         |0.716  |
|torch.float16|64        |64             |16     |512      |392.182                 |329.052                  |1.192                         |0.729  |
|torch.float16|64        |64             |8      |1024     |433.234                 |354.683                  |1.221                         |0.034  |
|torch.float16|64        |64             |8      |1024     |432.097                 |341.473                  |1.265                         |0.207  |
|torch.float16|64        |64             |8      |1024     |429.507                 |334.797                  |1.283                         |0.337  |
|torch.float16|64        |64             |8      |1024     |428.257                 |345.764                  |1.239                         |0.418  |
|torch.float16|64        |64             |8      |1024     |424.929                 |342.211                  |1.242                         |0.479  |
|torch.float16|64        |64             |8      |1024     |427.387                 |339.640                  |1.258                         |0.496  |
|torch.float16|64        |64             |8      |1024     |430.203                 |344.801                  |1.248                         |0.524  |
|torch.float16|64        |64             |8      |1024     |424.980                 |336.353                  |1.263                         |0.568  |
|torch.float16|64        |64             |8      |1024     |418.191                 |332.800                  |1.257                         |0.583  |
|torch.float16|64        |64             |8      |1024     |421.970                 |327.383                  |1.289                         |0.613  |
|torch.float16|64        |64             |8      |1024     |421.683                 |328.561                  |1.283                         |0.640  |
|torch.float16|64        |64             |8      |1024     |422.810                 |338.391                  |1.249                         |0.663  |
|torch.float16|64        |64             |8      |1024     |419.779                 |359.946                  |1.166                         |0.683  |
|torch.float16|64        |64             |8      |1024     |426.793                 |338.166                  |1.262                         |0.702  |
|torch.float16|64        |64             |8      |1024     |426.742                 |344.914                  |1.237                         |0.716  |
|torch.float16|64        |64             |8      |1024     |423.127                 |330.209                  |1.281                         |0.729  |
|torch.float16|64        |64             |16     |1024     |430.264                 |348.611                  |1.234                         |0.034  |
|torch.float16|64        |64             |16     |1024     |434.575                 |333.844                  |1.302                         |0.207  |
|torch.float16|64        |64             |16     |1024     |434.115                 |329.236                  |1.319                         |0.337  |
|torch.float16|64        |64             |16     |1024     |434.749                 |339.354                  |1.281                         |0.418  |
|torch.float16|64        |64             |16     |1024     |435.282                 |329.277                  |1.322                         |0.479  |
|torch.float16|64        |64             |16     |1024     |434.606                 |332.503                  |1.307                         |0.496  |
|torch.float16|64        |64             |16     |1024     |435.937                 |331.878                  |1.314                         |0.524  |
|torch.float16|64        |64             |16     |1024     |437.903                 |336.548                  |1.301                         |0.568  |
|torch.float16|64        |64             |16     |1024     |436.931                 |331.305                  |1.319                         |0.583  |
|torch.float16|64        |64             |16     |1024     |436.521                 |334.121                  |1.306                         |0.613  |
|torch.float16|64        |64             |16     |1024     |437.187                 |327.066                  |1.337                         |0.640  |
|torch.float16|64        |64             |16     |1024     |437.340                 |341.340                  |1.281                         |0.663  |
|torch.float16|64        |64             |16     |1024     |436.582                 |336.077                  |1.299                         |0.683  |
|torch.float16|64        |64             |16     |1024     |436.890                 |335.718                  |1.301                         |0.702  |
|torch.float16|64        |64             |16     |1024     |435.814                 |355.594                  |1.226                         |0.716  |
|torch.float16|64        |64             |16     |1024     |439.050                 |332.667                  |1.320                         |0.729  |
|torch.float16|64        |128            |8      |512      |412.232                 |327.199                  |1.260                         |0.033  |
|torch.float16|64        |128            |8      |512      |418.068                 |328.325                  |1.273                         |0.207  |
|torch.float16|64        |128            |8      |512      |415.150                 |322.939                  |1.286                         |0.334  |
|torch.float16|64        |128            |8      |512      |414.802                 |326.748                  |1.269                         |0.414  |
|torch.float16|64        |128            |8      |512      |413.921                 |328.151                  |1.261                         |0.475  |
|torch.float16|64        |128            |8      |512      |415.693                 |327.506                  |1.269                         |0.493  |
|torch.float16|64        |128            |8      |512      |416.358                 |323.482                  |1.287                         |0.523  |
|torch.float16|64        |128            |8      |512      |425.503                 |324.424                  |1.312                         |0.562  |
|torch.float16|64        |128            |8      |512      |419.809                 |333.783                  |1.258                         |0.579  |
|torch.float16|64        |128            |8      |512      |421.263                 |328.765                  |1.281                         |0.609  |
|torch.float16|64        |128            |8      |512      |450.273                 |327.158                  |1.376                         |0.636  |
|torch.float16|64        |128            |8      |512      |420.116                 |325.734                  |1.290                         |0.659  |
|torch.float16|64        |128            |8      |512      |419.123                 |331.459                  |1.264                         |0.679  |
|torch.float16|64        |128            |8      |512      |421.908                 |322.253                  |1.309                         |0.698  |
|torch.float16|64        |128            |8      |512      |418.601                 |331.858                  |1.261                         |0.713  |
|torch.float16|64        |128            |8      |512      |417.884                 |330.342                  |1.265                         |0.728  |
|torch.float16|64        |128            |16     |512      |531.896                 |324.628                  |1.638                         |0.033  |
|torch.float16|64        |128            |16     |512      |534.292                 |327.926                  |1.629                         |0.207  |
|torch.float16|64        |128            |16     |512      |536.893                 |322.478                  |1.665                         |0.334  |
|torch.float16|64        |128            |16     |512      |538.255                 |325.233                  |1.655                         |0.414  |
|torch.float16|64        |128            |16     |512      |539.443                 |337.664                  |1.598                         |0.475  |
|torch.float16|64        |128            |16     |512      |539.136                 |332.769                  |1.620                         |0.493  |
|torch.float16|64        |128            |16     |512      |540.150                 |332.728                  |1.623                         |0.523  |
|torch.float16|64        |128            |16     |512      |540.713                 |331.674                  |1.630                         |0.562  |
|torch.float16|64        |128            |16     |512      |541.542                 |324.209                  |1.670                         |0.579  |
|torch.float16|64        |128            |16     |512      |542.628                 |330.353                  |1.643                         |0.609  |
|torch.float16|64        |128            |16     |512      |542.689                 |332.728                  |1.631                         |0.636  |
|torch.float16|64        |128            |16     |512      |543.785                 |332.247                  |1.637                         |0.659  |
|torch.float16|64        |128            |16     |512      |545.014                 |329.892                  |1.652                         |0.679  |
|torch.float16|64        |128            |16     |512      |544.471                 |323.932                  |1.681                         |0.698  |
|torch.float16|64        |128            |16     |512      |544.092                 |323.543                  |1.682                         |0.713  |
|torch.float16|64        |128            |16     |512      |544.584                 |325.745                  |1.672                         |0.728  |
|torch.float16|64        |128            |8      |1024     |831.939                 |451.707                  |1.842                         |0.033  |
|torch.float16|64        |128            |8      |1024     |833.188                 |416.174                  |2.002                         |0.207  |
|torch.float16|64        |128            |8      |1024     |834.734                 |383.068                  |2.179                         |0.334  |
|torch.float16|64        |128            |8      |1024     |835.738                 |371.988                  |2.247                         |0.414  |
|torch.float16|64        |128            |8      |1024     |835.512                 |382.269                  |2.186                         |0.475  |
|torch.float16|64        |128            |8      |1024     |836.260                 |378.419                  |2.210                         |0.493  |
|torch.float16|64        |128            |8      |1024     |835.840                 |346.399                  |2.413                         |0.523  |
|torch.float16|64        |128            |8      |1024     |837.069                 |361.861                  |2.313                         |0.562  |
|torch.float16|64        |128            |8      |1024     |836.843                 |338.606                  |2.471                         |0.579  |
|torch.float16|64        |128            |8      |1024     |837.847                 |333.834                  |2.510                         |0.609  |
|torch.float16|64        |128            |8      |1024     |837.366                 |343.071                  |2.441                         |0.636  |
|torch.float16|64        |128            |8      |1024     |837.632                 |327.455                  |2.558                         |0.659  |
|torch.float16|64        |128            |8      |1024     |839.352                 |327.322                  |2.564                         |0.679  |
|torch.float16|64        |128            |8      |1024     |839.690                 |336.804                  |2.493                         |0.698  |
|torch.float16|64        |128            |8      |1024     |838.216                 |337.674                  |2.482                         |0.713  |
|torch.float16|64        |128            |8      |1024     |839.895                 |329.953                  |2.545                         |0.728  |
|torch.float16|64        |128            |16     |1024     |937.236                 |455.311                  |2.058                         |0.033  |
|torch.float16|64        |128            |16     |1024     |940.032                 |418.550                  |2.246                         |0.207  |
|torch.float16|64        |128            |16     |1024     |941.230                 |386.601                  |2.435                         |0.334  |
|torch.float16|64        |128            |16     |1024     |942.889                 |371.231                  |2.540                         |0.414  |
|torch.float16|64        |128            |16     |1024     |943.217                 |353.239                  |2.670                         |0.475  |
|torch.float16|64        |128            |16     |1024     |943.811                 |381.983                  |2.471                         |0.493  |
|torch.float16|64        |128            |16     |1024     |943.964                 |347.689                  |2.715                         |0.523  |
|torch.float16|64        |128            |16     |1024     |945.111                 |389.806                  |2.425                         |0.562  |
|torch.float16|64        |128            |16     |1024     |945.029                 |335.514                  |2.817                         |0.579  |
|torch.float16|64        |128            |16     |1024     |945.633                 |329.605                  |2.869                         |0.609  |
|torch.float16|64        |128            |16     |1024     |946.104                 |346.767                  |2.728                         |0.636  |
|torch.float16|64        |128            |16     |1024     |946.606                 |323.389                  |2.927                         |0.659  |
|torch.float16|64        |128            |16     |1024     |948.869                 |325.356                  |2.916                         |0.679  |
|torch.float16|64        |128            |16     |1024     |948.623                 |335.841                  |2.825                         |0.698  |
|torch.float16|64        |128            |16     |1024     |947.343                 |342.641                  |2.765                         |0.713  |
|torch.float16|64        |128            |16     |1024     |949.473                 |331.182                  |2.867                         |0.728  |
|torch.float16|64        |256            |8      |512      |990.208                 |357.724                  |2.768                         |0.033  |
|torch.float16|64        |256            |8      |512      |994.918                 |338.924                  |2.936                         |0.206  |
|torch.float16|64        |256            |8      |512      |999.741                 |323.697                  |3.089                         |0.333  |
|torch.float16|64        |256            |8      |512      |1002.127                |328.294                  |3.053                         |0.413  |
|torch.float16|64        |256            |8      |512      |1005.507                |333.763                  |3.013                         |0.475  |
|torch.float16|64        |256            |8      |512      |1006.551                |333.978                  |3.014                         |0.492  |
|torch.float16|64        |256            |8      |512      |1006.551                |332.974                  |3.023                         |0.521  |
|torch.float16|64        |256            |8      |512      |1008.425                |343.890                  |2.932                         |0.561  |
|torch.float16|64        |256            |8      |512      |1009.203                |331.356                  |3.046                         |0.577  |
|torch.float16|64        |256            |8      |512      |1010.166                |358.175                  |2.820                         |0.607  |
|torch.float16|64        |256            |8      |512      |1010.196                |332.554                  |3.038                         |0.633  |
|torch.float16|64        |256            |8      |512      |1011.016                |333.548                  |3.031                         |0.656  |
|torch.float16|64        |256            |8      |512      |1012.306                |332.206                  |3.047                         |0.677  |
|torch.float16|64        |256            |8      |512      |1013.883                |338.412                  |2.996                         |0.694  |
|torch.float16|64        |256            |8      |512      |1014.159                |334.397                  |3.033                         |0.710  |
|torch.float16|64        |256            |8      |512      |1014.979                |334.684                  |3.033                         |0.725  |
|torch.float16|64        |256            |16     |512      |1364.163                |414.751                  |3.289                         |0.033  |
|torch.float16|64        |256            |16     |512      |1377.751                |367.186                  |3.752                         |0.206  |
|torch.float16|64        |256            |16     |512      |1385.759                |344.074                  |4.027                         |0.333  |
|torch.float16|64        |256            |16     |512      |1388.800                |335.063                  |4.145                         |0.413  |
|torch.float16|64        |256            |16     |512      |1393.725                |333.271                  |4.182                         |0.475  |
|torch.float16|64        |256            |16     |512      |1394.391                |334.520                  |4.168                         |0.492  |
|torch.float16|64        |256            |16     |512      |1394.432                |333.496                  |4.181                         |0.521  |
|torch.float16|64        |256            |16     |512      |1398.815                |337.111                  |4.149                         |0.561  |
|torch.float16|64        |256            |16     |512      |1399.276                |335.032                  |4.177                         |0.577  |
|torch.float16|64        |256            |16     |512      |1401.784                |335.360                  |4.180                         |0.607  |
|torch.float16|64        |256            |16     |512      |1403.126                |330.742                  |4.242                         |0.633  |
|torch.float16|64        |256            |16     |512      |1404.457                |329.185                  |4.266                         |0.656  |
|torch.float16|64        |256            |16     |512      |1407.048                |330.220                  |4.261                         |0.677  |
|torch.float16|64        |256            |16     |512      |1410.017                |334.653                  |4.213                         |0.694  |
|torch.float16|64        |256            |16     |512      |1411.492                |337.265                  |4.185                         |0.710  |
|torch.float16|64        |256            |16     |512      |1413.366                |337.848                  |4.183                         |0.725  |
|torch.float16|64        |256            |8      |1024     |1784.668                |842.936                  |2.117                         |0.033  |
|torch.float16|64        |256            |8      |1024     |1784.228                |704.369                  |2.533                         |0.206  |
|torch.float16|64        |256            |8      |1024     |1781.996                |613.478                  |2.905                         |0.333  |
|torch.float16|64        |256            |8      |1024     |1783.480                |551.055                  |3.236                         |0.413  |
|torch.float16|64        |256            |8      |1024     |1780.234                |521.503                  |3.414                         |0.475  |
|torch.float16|64        |256            |8      |1024     |1777.777                |545.516                  |3.259                         |0.492  |
|torch.float16|64        |256            |8      |1024     |1772.995                |528.026                  |3.358                         |0.521  |
|torch.float16|64        |256            |8      |1024     |1773.681                |476.211                  |3.725                         |0.561  |
|torch.float16|64        |256            |8      |1024     |1775.821                |474.583                  |3.742                         |0.577  |
|torch.float16|64        |256            |8      |1024     |1776.384                |456.397                  |3.892                         |0.607  |
|torch.float16|64        |256            |8      |1024     |1777.480                |448.840                  |3.960                         |0.633  |
|torch.float16|64        |256            |8      |1024     |1777.644                |437.289                  |4.065                         |0.656  |
|torch.float16|64        |256            |8      |1024     |1778.801                |464.476                  |3.830                         |0.677  |
|torch.float16|64        |256            |8      |1024     |1779.886                |408.535                  |4.357                         |0.694  |
|torch.float16|64        |256            |8      |1024     |1780.675                |441.252                  |4.036                         |0.710  |
|torch.float16|64        |256            |8      |1024     |1782.487                |400.456                  |4.451                         |0.725  |
|torch.float16|64        |256            |16     |1024     |2151.793                |886.774                  |2.427                         |0.033  |
|torch.float16|64        |256            |16     |1024     |2165.381                |741.007                  |2.922                         |0.206  |
|torch.float16|64        |256            |16     |1024     |2166.467                |640.625                  |3.382                         |0.333  |
|torch.float16|64        |256            |16     |1024     |2170.583                |568.340                  |3.819                         |0.413  |
|torch.float16|64        |256            |16     |1024     |2168.371                |535.153                  |4.052                         |0.475  |
|torch.float16|64        |256            |16     |1024     |2169.334                |512.809                  |4.230                         |0.492  |
|torch.float16|64        |256            |16     |1024     |2167.787                |495.749                  |4.373                         |0.521  |
|torch.float16|64        |256            |16     |1024     |2166.262                |478.095                  |4.531                         |0.561  |
|torch.float16|64        |256            |16     |1024     |2168.484                |476.518                  |4.551                         |0.577  |
|torch.float16|64        |256            |16     |1024     |2171.269                |455.537                  |4.766                         |0.607  |
|torch.float16|64        |256            |16     |1024     |2172.672                |447.836                  |4.851                         |0.633  |
|torch.float16|64        |256            |16     |1024     |2173.747                |428.933                  |5.068                         |0.656  |
|torch.float16|64        |256            |16     |1024     |2175.877                |424.919                  |5.121                         |0.677  |
|torch.float16|64        |256            |16     |1024     |2178.939                |409.426                  |5.322                         |0.694  |
|torch.float16|64        |256            |16     |1024     |2180.977                |440.340                  |4.953                         |0.710  |
|torch.float16|64        |256            |16     |1024     |2183.352                |397.179                  |5.497                         |0.725  |
|torch.float16|64        |512            |8      |512      |2671.411                |796.959                  |3.352                         |0.033  |
|torch.float16|64        |512            |8      |512      |2695.875                |664.044                  |4.060                         |0.206  |
|torch.float16|64        |512            |8      |512      |2714.757                |554.650                  |4.895                         |0.332  |
|torch.float16|64        |512            |8      |512      |2720.768                |488.940                  |5.565                         |0.412  |
|torch.float16|64        |512            |8      |512      |2728.356                |441.641                  |6.178                         |0.473  |
|torch.float16|64        |512            |8      |512      |2734.572                |433.900                  |6.302                         |0.491  |
|torch.float16|64        |512            |8      |512      |2731.407                |417.208                  |6.547                         |0.519  |
|torch.float16|64        |512            |8      |512      |2738.360                |396.339                  |6.909                         |0.560  |
|torch.float16|64        |512            |8      |512      |2743.306                |393.738                  |6.967                         |0.576  |
|torch.float16|64        |512            |8      |512      |2750.689                |383.447                  |7.174                         |0.606  |
|torch.float16|64        |512            |8      |512      |2751.549                |379.464                  |7.251                         |0.632  |
|torch.float16|64        |512            |8      |512      |2755.246                |380.846                  |7.235                         |0.655  |
|torch.float16|64        |512            |8      |512      |2758.769                |416.737                  |6.620                         |0.676  |
|torch.float16|64        |512            |8      |512      |2766.766                |383.027                  |7.223                         |0.693  |
|torch.float16|64        |512            |8      |512      |2768.681                |382.935                  |7.230                         |0.709  |
|torch.float16|64        |512            |8      |512      |2771.292                |383.099                  |7.234                         |0.724  |
|torch.float16|64        |512            |16     |512      |4186.501                |1092.475                 |3.832                         |0.033  |
|torch.float16|64        |512            |16     |512      |4226.959                |914.862                  |4.620                         |0.206  |
|torch.float16|64        |512            |16     |512      |4258.161                |751.954                  |5.663                         |0.332  |
|torch.float16|64        |512            |16     |512      |4283.382                |648.663                  |6.603                         |0.412  |
|torch.float16|64        |512            |16     |512      |4301.332                |574.986                  |7.481                         |0.473  |
|torch.float16|64        |512            |16     |512      |4310.743                |563.159                  |7.655                         |0.491  |
|torch.float16|64        |512            |16     |512      |4312.729                |529.541                  |8.144                         |0.519  |
|torch.float16|64        |512            |16     |512      |4326.533                |483.011                  |8.957                         |0.560  |
|torch.float16|64        |512            |16     |512      |4333.138                |472.453                  |9.172                         |0.576  |
|torch.float16|64        |512            |16     |512      |4345.467                |434.463                  |10.002                        |0.606  |
|torch.float16|64        |512            |16     |512      |4353.004                |412.457                  |10.554                        |0.632  |
|torch.float16|64        |512            |16     |512      |4363.049                |380.058                  |11.480                        |0.655  |
|torch.float16|64        |512            |16     |512      |4369.879                |360.335                  |12.127                        |0.676  |
|torch.float16|64        |512            |16     |512      |4381.675                |347.576                  |12.606                        |0.693  |
|torch.float16|64        |512            |16     |512      |4386.222                |338.156                  |12.971                        |0.709  |
|torch.float16|64        |512            |16     |512      |4393.032                |335.718                  |13.085                        |0.724  |
|torch.float16|64        |512            |8      |1024     |4159.345                |1896.264                 |2.193                         |0.033  |
|torch.float16|64        |512            |8      |1024     |4173.435                |1572.004                 |2.655                         |0.206  |
|torch.float16|64        |512            |8      |1024     |4190.331                |1329.736                 |3.151                         |0.332  |
|torch.float16|64        |512            |8      |1024     |4196.147                |1161.359                 |3.613                         |0.412  |
|torch.float16|64        |512            |8      |1024     |4199.086                |992.901                  |4.229                         |0.473  |
|torch.float16|64        |512            |8      |1024     |4199.547                |996.731                  |4.213                         |0.491  |
|torch.float16|64        |512            |8      |1024     |4197.263                |943.329                  |4.449                         |0.519  |
|torch.float16|64        |512            |8      |1024     |4200.417                |858.706                  |4.892                         |0.560  |
|torch.float16|64        |512            |8      |1024     |4206.346                |835.881                  |5.032                         |0.576  |
|torch.float16|64        |512            |8      |1024     |4212.142                |803.697                  |5.241                         |0.606  |
|torch.float16|64        |512            |8      |1024     |4215.193                |714.291                  |5.901                         |0.632  |
|torch.float16|64        |512            |8      |1024     |4214.303                |667.853                  |6.310                         |0.655  |
|torch.float16|64        |512            |8      |1024     |4218.071                |622.725                  |6.774                         |0.676  |
|torch.float16|64        |512            |8      |1024     |4221.440                |587.704                  |7.183                         |0.693  |
|torch.float16|64        |512            |8      |1024     |4222.126                |562.084                  |7.512                         |0.709  |
|torch.float16|64        |512            |8      |1024     |4223.120                |571.556                  |7.389                         |0.724  |
|torch.float16|64        |512            |16     |1024     |5685.309                |2022.072                 |2.812                         |0.033  |
|torch.float16|64        |512            |16     |1024     |5721.139                |1690.132                 |3.385                         |0.206  |
|torch.float16|64        |512            |16     |1024     |5749.248                |1410.929                 |4.075                         |0.332  |
|torch.float16|64        |512            |16     |1024     |5768.898                |1227.848                 |4.698                         |0.412  |
|torch.float16|64        |512            |16     |1024     |5776.445                |1046.743                 |5.518                         |0.473  |
|torch.float16|64        |512            |16     |1024     |5791.785                |1058.847                 |5.470                         |0.491  |
|torch.float16|64        |512            |16     |1024     |5786.440                |991.007                  |5.839                         |0.519  |
|torch.float16|64        |512            |16     |1024     |5800.663                |907.971                  |6.389                         |0.560  |
|torch.float16|64        |512            |16     |1024     |5801.216                |906.988                  |6.396                         |0.576  |
|torch.float16|64        |512            |16     |1024     |5821.317                |801.864                  |7.260                         |0.606  |
|torch.float16|64        |512            |16     |1024     |5827.154                |756.101                  |7.707                         |0.632  |
|torch.float16|64        |512            |16     |1024     |5833.974                |706.345                  |8.259                         |0.655  |
|torch.float16|64        |512            |16     |1024     |5841.981                |655.032                  |8.919                         |0.676  |
|torch.float16|64        |512            |16     |1024     |5847.153                |615.332                  |9.502                         |0.693  |
|torch.float16|64        |512            |16     |1024     |5858.888                |586.148                  |9.996                         |0.709  |
|torch.float16|64        |512            |16     |1024     |5859.819                |556.698                  |10.526                        |0.724  |
|torch.float16|128       |64             |8      |512      |432.753                 |430.592                  |1.005                         |0.054  |
|torch.float16|128       |64             |8      |512      |436.070                 |463.503                  |0.941                         |0.219  |
|torch.float16|128       |64             |8      |512      |430.551                 |425.103                  |1.013                         |0.322  |
|torch.float16|128       |64             |8      |512      |431.852                 |427.325                  |1.011                         |0.401  |
|torch.float16|128       |64             |8      |512      |433.500                 |425.882                  |1.018                         |0.464  |
|torch.float16|128       |64             |8      |512      |432.742                 |428.298                  |1.010                         |0.477  |
|torch.float16|128       |64             |8      |512      |433.367                 |431.104                  |1.005                         |0.515  |
|torch.float16|128       |64             |8      |512      |436.582                 |429.537                  |1.016                         |0.554  |
|torch.float16|128       |64             |8      |512      |438.999                 |427.735                  |1.026                         |0.580  |
|torch.float16|128       |64             |8      |512      |432.220                 |427.807                  |1.010                         |0.610  |
|torch.float16|128       |64             |8      |512      |432.015                 |428.964                  |1.007                         |0.638  |
|torch.float16|128       |64             |8      |512      |431.657                 |425.175                  |1.015                         |0.662  |
|torch.float16|128       |64             |8      |512      |432.353                 |425.533                  |1.016                         |0.682  |
|torch.float16|128       |64             |8      |512      |434.616                 |420.731                  |1.033                         |0.700  |
|torch.float16|128       |64             |8      |512      |429.875                 |424.950                  |1.012                         |0.716  |
|torch.float16|128       |64             |8      |512      |431.073                 |421.990                  |1.022                         |0.731  |
|torch.float16|128       |64             |16     |512      |430.510                 |429.261                  |1.003                         |0.054  |
|torch.float16|128       |64             |16     |512      |433.633                 |426.025                  |1.018                         |0.219  |
|torch.float16|128       |64             |16     |512      |435.692                 |432.620                  |1.007                         |0.322  |
|torch.float16|128       |64             |16     |512      |472.371                 |427.500                  |1.105                         |0.401  |
|torch.float16|128       |64             |16     |512      |435.958                 |428.780                  |1.017                         |0.464  |
|torch.float16|128       |64             |16     |512      |433.521                 |430.111                  |1.008                         |0.477  |
|torch.float16|128       |64             |16     |512      |434.135                 |422.748                  |1.027                         |0.515  |
|torch.float16|128       |64             |16     |512      |433.961                 |423.782                  |1.024                         |0.554  |
|torch.float16|128       |64             |16     |512      |435.210                 |426.383                  |1.021                         |0.580  |
|torch.float16|128       |64             |16     |512      |435.896                 |427.858                  |1.019                         |0.610  |
|torch.float16|128       |64             |16     |512      |437.238                 |426.148                  |1.026                         |0.638  |
|torch.float16|128       |64             |16     |512      |435.180                 |430.316                  |1.011                         |0.662  |
|torch.float16|128       |64             |16     |512      |437.248                 |428.964                  |1.019                         |0.682  |
|torch.float16|128       |64             |16     |512      |438.743                 |430.080                  |1.020                         |0.700  |
|torch.float16|128       |64             |16     |512      |437.750                 |425.912                  |1.028                         |0.716  |
|torch.float16|128       |64             |16     |512      |437.658                 |423.229                  |1.034                         |0.731  |
|torch.float16|128       |64             |8      |1024     |782.100                 |502.671                  |1.556                         |0.054  |
|torch.float16|128       |64             |8      |1024     |782.326                 |474.870                  |1.647                         |0.219  |
|torch.float16|128       |64             |8      |1024     |784.087                 |448.870                  |1.747                         |0.322  |
|torch.float16|128       |64             |8      |1024     |783.217                 |441.897                  |1.772                         |0.401  |
|torch.float16|128       |64             |8      |1024     |783.514                 |435.425                  |1.799                         |0.464  |
|torch.float16|128       |64             |8      |1024     |782.899                 |434.340                  |1.803                         |0.477  |
|torch.float16|128       |64             |8      |1024     |784.415                 |448.266                  |1.750                         |0.515  |
|torch.float16|128       |64             |8      |1024     |785.050                 |445.901                  |1.761                         |0.554  |
|torch.float16|128       |64             |8      |1024     |784.742                 |424.489                  |1.849                         |0.580  |
|torch.float16|128       |64             |8      |1024     |784.015                 |425.452                  |1.843                         |0.610  |
|torch.float16|128       |64             |8      |1024     |785.080                 |434.186                  |1.808                         |0.638  |
|torch.float16|128       |64             |8      |1024     |784.763                 |422.185                  |1.859                         |0.662  |
|torch.float16|128       |64             |8      |1024     |786.760                 |413.010                  |1.905                         |0.682  |
|torch.float16|128       |64             |8      |1024     |784.937                 |425.032                  |1.847                         |0.700  |
|torch.float16|128       |64             |8      |1024     |785.459                 |438.825                  |1.790                         |0.716  |
|torch.float16|128       |64             |8      |1024     |787.149                 |425.247                  |1.851                         |0.731  |
|torch.float16|128       |64             |16     |1024     |833.884                 |500.275                  |1.667                         |0.054  |
|torch.float16|128       |64             |16     |1024     |835.973                 |466.954                  |1.790                         |0.219  |
|torch.float16|128       |64             |16     |1024     |837.028                 |488.018                  |1.715                         |0.322  |
|torch.float16|128       |64             |16     |1024     |838.328                 |434.104                  |1.931                         |0.401  |
|torch.float16|128       |64             |16     |1024     |838.881                 |424.264                  |1.977                         |0.464  |
|torch.float16|128       |64             |16     |1024     |838.779                 |416.328                  |2.015                         |0.477  |
|torch.float16|128       |64             |16     |1024     |839.066                 |445.082                  |1.885                         |0.515  |
|torch.float16|128       |64             |16     |1024     |839.967                 |432.282                  |1.943                         |0.554  |
|torch.float16|128       |64             |16     |1024     |839.649                 |412.549                  |2.035                         |0.580  |
|torch.float16|128       |64             |16     |1024     |840.581                 |415.447                  |2.023                         |0.610  |
|torch.float16|128       |64             |16     |1024     |840.264                 |431.350                  |1.948                         |0.638  |
|torch.float16|128       |64             |16     |1024     |841.011                 |465.316                  |1.807                         |0.662  |
|torch.float16|128       |64             |16     |1024     |842.844                 |415.631                  |2.028                         |0.682  |
|torch.float16|128       |64             |16     |1024     |841.769                 |432.323                  |1.947                         |0.700  |
|torch.float16|128       |64             |16     |1024     |842.527                 |431.647                  |1.952                         |0.716  |
|torch.float16|128       |64             |16     |1024     |843.848                 |423.229                  |1.994                         |0.731  |
|torch.float16|128       |128            |8      |512      |799.437                 |467.814                  |1.709                         |0.054  |
|torch.float16|128       |128            |8      |512      |803.420                 |419.451                  |1.915                         |0.219  |
|torch.float16|128       |128            |8      |512      |803.154                 |416.154                  |1.930                         |0.320  |
|torch.float16|128       |128            |8      |512      |805.059                 |412.600                  |1.951                         |0.397  |
|torch.float16|128       |128            |8      |512      |806.994                 |421.601                  |1.914                         |0.460  |
|torch.float16|128       |128            |8      |512      |805.980                 |424.909                  |1.897                         |0.475  |
|torch.float16|128       |128            |8      |512      |805.990                 |417.362                  |1.931                         |0.513  |
|torch.float16|128       |128            |8      |512      |806.707                 |420.987                  |1.916                         |0.550  |
|torch.float16|128       |128            |8      |512      |807.127                 |469.166                  |1.720                         |0.576  |
|torch.float16|128       |128            |8      |512      |807.680                 |419.072                  |1.927                         |0.607  |
|torch.float16|128       |128            |8      |512      |807.946                 |420.403                  |1.922                         |0.634  |
|torch.float16|128       |128            |8      |512      |808.602                 |416.543                  |1.941                         |0.658  |
|torch.float16|128       |128            |8      |512      |809.329                 |413.583                  |1.957                         |0.678  |
|torch.float16|128       |128            |8      |512      |809.800                 |458.414                  |1.767                         |0.697  |
|torch.float16|128       |128            |8      |512      |809.871                 |414.792                  |1.952                         |0.713  |
|torch.float16|128       |128            |8      |512      |810.414                 |420.536                  |1.927                         |0.728  |
|torch.float16|128       |128            |16     |512      |993.638                 |419.430                  |2.369                         |0.054  |
|torch.float16|128       |128            |16     |512      |1003.919                |416.717                  |2.409                         |0.219  |
|torch.float16|128       |128            |16     |512      |1005.722                |417.116                  |2.411                         |0.320  |
|torch.float16|128       |128            |16     |512      |1007.923                |417.649                  |2.413                         |0.397  |
|torch.float16|128       |128            |16     |512      |1012.214                |415.969                  |2.433                         |0.460  |
|torch.float16|128       |128            |16     |512      |1011.098                |423.956                  |2.385                         |0.475  |
|torch.float16|128       |128            |16     |512      |1011.333                |415.580                  |2.434                         |0.513  |
|torch.float16|128       |128            |16     |512      |1013.043                |418.877                  |2.418                         |0.550  |
|torch.float16|128       |128            |16     |512      |1013.975                |424.919                  |2.386                         |0.576  |
|torch.float16|128       |128            |16     |512      |1015.716                |415.928                  |2.442                         |0.607  |
|torch.float16|128       |128            |16     |512      |1017.170                |423.619                  |2.401                         |0.634  |
|torch.float16|128       |128            |16     |512      |1018.112                |412.426                  |2.469                         |0.658  |
|torch.float16|128       |128            |16     |512      |1018.911                |418.345                  |2.436                         |0.678  |
|torch.float16|128       |128            |16     |512      |1018.890                |415.119                  |2.454                         |0.697  |
|torch.float16|128       |128            |16     |512      |1019.771                |417.423                  |2.443                         |0.713  |
|torch.float16|128       |128            |16     |512      |1023.498                |419.451                  |2.440                         |0.728  |
|torch.float16|128       |128            |8      |1024     |1589.658                |764.672                  |2.079                         |0.054  |
|torch.float16|128       |128            |8      |1024     |1592.750                |655.555                  |2.430                         |0.219  |
|torch.float16|128       |128            |8      |1024     |1590.118                |593.766                  |2.678                         |0.320  |
|torch.float16|128       |128            |8      |1024     |1588.081                |559.852                  |2.837                         |0.397  |
|torch.float16|128       |128            |8      |1024     |1581.425                |533.258                  |2.966                         |0.460  |
|torch.float16|128       |128            |8      |1024     |1576.284                |528.609                  |2.982                         |0.475  |
|torch.float16|128       |128            |8      |1024     |1576.243                |502.364                  |3.138                         |0.513  |
|torch.float16|128       |128            |8      |1024     |1576.725                |487.209                  |3.236                         |0.550  |
|torch.float16|128       |128            |8      |1024     |1577.943                |492.472                  |3.204                         |0.576  |
|torch.float16|128       |128            |8      |1024     |1579.305                |478.075                  |3.303                         |0.607  |
|torch.float16|128       |128            |8      |1024     |1578.373                |458.373                  |3.443                         |0.634  |
|torch.float16|128       |128            |8      |1024     |1579.100                |451.410                  |3.498                         |0.658  |
|torch.float16|128       |128            |8      |1024     |1579.428                |445.624                  |3.544                         |0.678  |
|torch.float16|128       |128            |8      |1024     |1580.657                |442.409                  |3.573                         |0.697  |
|torch.float16|128       |128            |8      |1024     |1580.800                |461.650                  |3.424                         |0.713  |
|torch.float16|128       |128            |8      |1024     |1581.404                |426.158                  |3.711                         |0.728  |
|torch.float16|128       |128            |16     |1024     |1776.732                |776.724                  |2.287                         |0.054  |
|torch.float16|128       |128            |16     |1024     |1787.310                |669.563                  |2.669                         |0.219  |
|torch.float16|128       |128            |16     |1024     |1785.293                |606.546                  |2.943                         |0.320  |
|torch.float16|128       |128            |16     |1024     |1779.415                |564.040                  |3.155                         |0.397  |
|torch.float16|128       |128            |16     |1024     |1775.135                |540.140                  |3.286                         |0.460  |
|torch.float16|128       |128            |16     |1024     |1774.940                |534.180                  |3.323                         |0.475  |
|torch.float16|128       |128            |16     |1024     |1773.363                |504.566                  |3.515                         |0.513  |
|torch.float16|128       |128            |16     |1024     |1774.633                |496.189                  |3.577                         |0.550  |
|torch.float16|128       |128            |16     |1024     |1777.428                |494.991                  |3.591                         |0.576  |
|torch.float16|128       |128            |16     |1024     |1779.077                |474.051                  |3.753                         |0.607  |
|torch.float16|128       |128            |16     |1024     |1779.138                |473.262                  |3.759                         |0.634  |
|torch.float16|128       |128            |16     |1024     |1779.855                |453.366                  |3.926                         |0.658  |
|torch.float16|128       |128            |16     |1024     |1780.613                |451.205                  |3.946                         |0.678  |
|torch.float16|128       |128            |16     |1024     |1782.108                |448.963                  |3.969                         |0.697  |
|torch.float16|128       |128            |16     |1024     |1783.132                |463.596                  |3.846                         |0.713  |
|torch.float16|128       |128            |16     |1024     |1783.511                |433.203                  |4.117                         |0.728  |
|torch.float16|128       |256            |8      |512      |1899.745                |590.520                  |3.217                         |0.054  |
|torch.float16|128       |256            |8      |512      |1897.789                |508.887                  |3.729                         |0.218  |
|torch.float16|128       |256            |8      |512      |1898.680                |469.617                  |4.043                         |0.320  |
|torch.float16|128       |256            |8      |512      |1903.841                |448.072                  |4.249                         |0.396  |
|torch.float16|128       |256            |8      |512      |1911.572                |434.964                  |4.395                         |0.459  |
|torch.float16|128       |256            |8      |512      |1913.313                |437.012                  |4.378                         |0.473  |
|torch.float16|128       |256            |8      |512      |1912.535                |424.806                  |4.502                         |0.511  |
|torch.float16|128       |256            |8      |512      |1917.706                |421.222                  |4.553                         |0.549  |
|torch.float16|128       |256            |8      |512      |1919.222                |423.772                  |4.529                         |0.574  |
|torch.float16|128       |256            |8      |512      |1917.747                |426.721                  |4.494                         |0.605  |
|torch.float16|128       |256            |8      |512      |1922.857                |423.660                  |4.539                         |0.632  |
|torch.float16|128       |256            |8      |512      |1921.290                |421.243                  |4.561                         |0.655  |
|torch.float16|128       |256            |8      |512      |1923.686                |418.499                  |4.597                         |0.676  |
|torch.float16|128       |256            |8      |512      |1929.288                |428.780                  |4.499                         |0.694  |
|torch.float16|128       |256            |8      |512      |1930.517                |432.609                  |4.462                         |0.711  |
|torch.float16|128       |256            |8      |512      |1932.370                |423.813                  |4.559                         |0.726  |
|torch.float16|128       |256            |16     |512      |2646.425                |743.465                  |3.560                         |0.054  |
|torch.float16|128       |256            |16     |512      |2661.356                |635.331                  |4.189                         |0.218  |
|torch.float16|128       |256            |16     |512      |2674.514                |570.368                  |4.689                         |0.320  |
|torch.float16|128       |256            |16     |512      |2677.576                |512.973                  |5.220                         |0.396  |
|torch.float16|128       |256            |16     |512      |2690.488                |472.484                  |5.694                         |0.459  |
|torch.float16|128       |256            |16     |512      |2694.257                |464.374                  |5.802                         |0.473  |
|torch.float16|128       |256            |16     |512      |2696.428                |442.563                  |6.093                         |0.511  |
|torch.float16|128       |256            |16     |512      |2705.142                |424.878                  |6.367                         |0.549  |
|torch.float16|128       |256            |16     |512      |2709.412                |432.783                  |6.260                         |0.574  |
|torch.float16|128       |256            |16     |512      |2710.221                |425.513                  |6.369                         |0.605  |
|torch.float16|128       |256            |16     |512      |2716.529                |422.973                  |6.422                         |0.632  |
|torch.float16|128       |256            |16     |512      |2717.839                |428.503                  |6.343                         |0.655  |
|torch.float16|128       |256            |16     |512      |2722.857                |423.598                  |6.428                         |0.676  |
|torch.float16|128       |256            |16     |512      |2731.244                |420.219                  |6.500                         |0.694  |
|torch.float16|128       |256            |16     |512      |2733.885                |422.011                  |6.478                         |0.711  |
|torch.float16|128       |256            |16     |512      |2737.265                |424.765                  |6.444                         |0.726  |
|torch.float16|128       |256            |8      |1024     |3376.732                |1533.215                 |2.202                         |0.054  |
|torch.float16|128       |256            |8      |1024     |3382.815                |1280.901                 |2.641                         |0.218  |
|torch.float16|128       |256            |8      |1024     |3383.501                |1122.365                 |3.015                         |0.320  |
|torch.float16|128       |256            |8      |1024     |3383.839                |1038.674                 |3.258                         |0.396  |
|torch.float16|128       |256            |8      |1024     |3385.027                |936.919                  |3.613                         |0.459  |
|torch.float16|128       |256            |8      |1024     |3385.620                |869.325                  |3.895                         |0.473  |
|torch.float16|128       |256            |8      |1024     |3381.279                |842.578                  |4.013                         |0.511  |
|torch.float16|128       |256            |8      |1024     |3381.678                |787.896                  |4.292                         |0.549  |
|torch.float16|128       |256            |8      |1024     |3383.368                |742.185                  |4.559                         |0.574  |
|torch.float16|128       |256            |8      |1024     |3384.238                |698.358                  |4.846                         |0.605  |
|torch.float16|128       |256            |8      |1024     |3388.754                |659.077                  |5.142                         |0.632  |
|torch.float16|128       |256            |8      |1024     |3383.562                |616.202                  |5.491                         |0.655  |
|torch.float16|128       |256            |8      |1024     |3381.647                |588.268                  |5.748                         |0.676  |
|torch.float16|128       |256            |8      |1024     |3385.047                |572.897                  |5.909                         |0.694  |
|torch.float16|128       |256            |8      |1024     |3379.937                |544.840                  |6.204                         |0.711  |
|torch.float16|128       |256            |8      |1024     |3382.426                |537.713                  |6.290                         |0.726  |
|torch.float16|128       |256            |16     |1024     |4135.414                |1606.994                 |2.573                         |0.054  |
|torch.float16|128       |256            |16     |1024     |4150.968                |1337.293                 |3.104                         |0.218  |
|torch.float16|128       |256            |16     |1024     |4157.153                |1173.586                 |3.542                         |0.320  |
|torch.float16|128       |256            |16     |1024     |4155.238                |1082.931                 |3.837                         |0.396  |
|torch.float16|128       |256            |16     |1024     |4163.379                |973.619                  |4.276                         |0.459  |
|torch.float16|128       |256            |16     |1024     |4165.345                |916.245                  |4.546                         |0.473  |
|torch.float16|128       |256            |16     |1024     |4162.386                |874.476                  |4.760                         |0.511  |
|torch.float16|128       |256            |16     |1024     |4177.019                |817.347                  |5.110                         |0.549  |
|torch.float16|128       |256            |16     |1024     |4172.892                |766.198                  |5.446                         |0.574  |
|torch.float16|128       |256            |16     |1024     |4174.838                |714.342                  |5.844                         |0.605  |
|torch.float16|128       |256            |16     |1024     |4187.822                |678.185                  |6.175                         |0.632  |
|torch.float16|128       |256            |16     |1024     |4179.743                |635.197                  |6.580                         |0.655  |
|torch.float16|128       |256            |16     |1024     |4183.941                |601.897                  |6.951                         |0.676  |
|torch.float16|128       |256            |16     |1024     |4189.757                |577.843                  |7.251                         |0.694  |
|torch.float16|128       |256            |16     |1024     |4186.931                |554.772                  |7.547                         |0.711  |
|torch.float16|128       |256            |16     |1024     |4193.055                |542.372                  |7.731                         |0.726  |
|torch.float16|128       |512            |8      |512      |5230.090                |1492.900                 |3.503                         |0.054  |
|torch.float16|128       |512            |8      |512      |5269.913                |1211.003                 |4.352                         |0.218  |
|torch.float16|128       |512            |8      |512      |5297.234                |1049.416                 |5.048                         |0.319  |
|torch.float16|128       |512            |8      |512      |5302.292                |931.543                  |5.692                         |0.395  |
|torch.float16|128       |512            |8      |512      |5321.953                |811.008                  |6.562                         |0.458  |
|torch.float16|128       |512            |8      |512      |5327.811                |791.101                  |6.735                         |0.472  |
|torch.float16|128       |512            |8      |512      |5328.374                |726.129                  |7.338                         |0.510  |
|torch.float16|128       |512            |8      |512      |5346.611                |669.491                  |7.986                         |0.548  |
|torch.float16|128       |512            |8      |512      |5351.445                |631.173                  |8.479                         |0.573  |
|torch.float16|128       |512            |8      |512      |5361.398                |583.547                  |9.188                         |0.604  |
|torch.float16|128       |512            |8      |512      |5368.033                |553.318                  |9.702                         |0.631  |
|torch.float16|128       |512            |8      |512      |5370.921                |519.496                  |10.339                        |0.654  |
|torch.float16|128       |512            |8      |512      |5377.884                |500.951                  |10.735                        |0.675  |
|torch.float16|128       |512            |8      |512      |5385.800                |485.202                  |11.100                        |0.693  |
|torch.float16|128       |512            |8      |512      |5389.722                |475.136                  |11.344                        |0.710  |
|torch.float16|128       |512            |8      |512      |5396.480                |470.589                  |11.467                        |0.724  |
|torch.float16|128       |512            |16     |512      |8248.853                |2090.906                 |3.945                         |0.054  |
|torch.float16|128       |512            |16     |512      |8328.550                |1675.090                 |4.972                         |0.218  |
|torch.float16|128       |512            |16     |512      |8374.026                |1445.530                 |5.793                         |0.319  |
|torch.float16|128       |512            |16     |512      |8411.402                |1271.460                 |6.616                         |0.395  |
|torch.float16|128       |512            |16     |512      |8451.953                |1139.241                 |7.419                         |0.458  |
|torch.float16|128       |512            |16     |512      |8457.410                |1075.978                 |7.860                         |0.472  |
|torch.float16|128       |512            |16     |512      |8474.153                |983.839                  |8.613                         |0.510  |
|torch.float16|128       |512            |16     |512      |8498.176                |900.178                  |9.441                         |0.548  |
|torch.float16|128       |512            |16     |512      |8508.600                |841.359                  |10.113                        |0.573  |
|torch.float16|128       |512            |16     |512      |8529.725                |776.550                  |10.984                        |0.604  |
|torch.float16|128       |512            |16     |512      |8548.506                |711.905                  |12.008                        |0.631  |
|torch.float16|128       |512            |16     |512      |8568.218                |688.312                  |12.448                        |0.654  |
|torch.float16|128       |512            |16     |512      |8585.093                |601.989                  |14.261                        |0.675  |
|torch.float16|128       |512            |16     |512      |8603.187                |560.527                  |15.348                        |0.693  |
|torch.float16|128       |512            |16     |512      |8613.028                |582.318                  |14.791                        |0.710  |
|torch.float16|128       |512            |16     |512      |8628.101                |509.839                  |16.923                        |0.724  |
|torch.float16|128       |512            |8      |1024     |8212.961                |3695.145                 |2.223                         |0.054  |
|torch.float16|128       |512            |8      |1024     |8249.723                |3009.116                 |2.742                         |0.218  |
|torch.float16|128       |512            |8      |1024     |8256.409                |2586.429                 |3.192                         |0.319  |
|torch.float16|128       |512            |8      |1024     |8262.021                |2288.189                 |3.611                         |0.395  |
|torch.float16|128       |512            |8      |1024     |8275.466                |2009.928                 |4.117                         |0.458  |
|torch.float16|128       |512            |8      |1024     |8280.074                |1958.554                 |4.228                         |0.472  |
|torch.float16|128       |512            |8      |1024     |8296.612                |1793.843                 |4.625                         |0.510  |
|torch.float16|128       |512            |8      |1024     |8302.151                |1643.715                 |5.051                         |0.548  |
|torch.float16|128       |512            |8      |1024     |8323.420                |1546.035                 |5.384                         |0.573  |
|torch.float16|128       |512            |8      |1024     |8323.901                |1420.278                 |5.861                         |0.604  |
|torch.float16|128       |512            |8      |1024     |8325.560                |1356.165                 |6.139                         |0.631  |
|torch.float16|128       |512            |8      |1024     |8331.837                |1268.214                 |6.570                         |0.654  |
|torch.float16|128       |512            |8      |1024     |8337.142                |1178.778                 |7.073                         |0.675  |
|torch.float16|128       |512            |8      |1024     |8344.515                |1062.830                 |7.851                         |0.693  |
|torch.float16|128       |512            |8      |1024     |8343.306                |1049.088                 |7.953                         |0.710  |
|torch.float16|128       |512            |8      |1024     |8353.823                |996.557                  |8.383                         |0.724  |
|torch.float16|128       |512            |16     |1024     |11258.972               |3895.388                 |2.890                         |0.054  |
|torch.float16|128       |512            |16     |1024     |11333.079               |3196.897                 |3.545                         |0.218  |
|torch.float16|128       |512            |16     |1024     |11371.335               |2756.925                 |4.125                         |0.319  |
|torch.float16|128       |512            |16     |1024     |11399.045               |2433.311                 |4.685                         |0.395  |
|torch.float16|128       |512            |16     |1024     |11435.858               |2113.802                 |5.410                         |0.458  |
|torch.float16|128       |512            |16     |1024     |11441.572               |2059.162                 |5.556                         |0.472  |
|torch.float16|128       |512            |16     |1024     |11462.892               |1891.502                 |6.060                         |0.510  |
|torch.float16|128       |512            |16     |1024     |11486.874               |1731.789                 |6.633                         |0.548  |
|torch.float16|128       |512            |16     |1024     |11507.006               |1623.337                 |7.088                         |0.573  |
|torch.float16|128       |512            |16     |1024     |11517.778               |1498.829                 |7.685                         |0.604  |
|torch.float16|128       |512            |16     |1024     |11531.776               |1430.251                 |8.063                         |0.631  |
|torch.float16|128       |512            |16     |1024     |11551.068               |1320.468                 |8.748                         |0.654  |
|torch.float16|128       |512            |16     |1024     |11569.009               |1231.606                 |9.393                         |0.675  |
|torch.float16|128       |512            |16     |1024     |11590.492               |1107.169                 |10.469                        |0.693  |
|torch.float16|128       |512            |16     |1024     |11593.707               |1092.536                 |10.612                        |0.710  |
|torch.float16|128       |512            |16     |1024     |11609.242               |1039.278                 |11.170                        |0.724  |
|torch.float16|256       |64             |8      |512      |693.473                 |581.100                  |1.193                         |0.041  |
|torch.float16|256       |64             |8      |512      |698.255                 |579.154                  |1.206                         |0.201  |
|torch.float16|256       |64             |8      |512      |698.849                 |595.702                  |1.173                         |0.306  |
|torch.float16|256       |64             |8      |512      |702.669                 |585.605                  |1.200                         |0.390  |
|torch.float16|256       |64             |8      |512      |701.901                 |581.642                  |1.207                         |0.453  |
|torch.float16|256       |64             |8      |512      |702.474                 |579.092                  |1.213                         |0.522  |
|torch.float16|256       |64             |8      |512      |702.362                 |584.643                  |1.201                         |0.564  |
|torch.float16|256       |64             |8      |512      |703.683                 |579.881                  |1.213                         |0.596  |
|torch.float16|256       |64             |8      |512      |702.904                 |579.850                  |1.212                         |0.622  |
|torch.float16|256       |64             |8      |512      |704.614                 |591.493                  |1.191                         |0.634  |
|torch.float16|256       |64             |8      |512      |703.959                 |577.638                  |1.219                         |0.661  |
|torch.float16|256       |64             |8      |512      |705.546                 |588.431                  |1.199                         |0.683  |
|torch.float16|256       |64             |8      |512      |704.440                 |579.963                  |1.215                         |0.701  |
|torch.float16|256       |64             |8      |512      |705.597                 |585.359                  |1.205                         |0.719  |
|torch.float16|256       |64             |8      |512      |704.481                 |579.011                  |1.217                         |0.734  |
|torch.float16|256       |64             |8      |512      |705.710                 |580.884                  |1.215                         |0.748  |
|torch.float16|256       |64             |16     |512      |808.929                 |592.445                  |1.365                         |0.041  |
|torch.float16|256       |64             |16     |512      |816.906                 |578.017                  |1.413                         |0.201  |
|torch.float16|256       |64             |16     |512      |819.108                 |588.995                  |1.391                         |0.306  |
|torch.float16|256       |64             |16     |512      |824.146                 |579.686                  |1.422                         |0.390  |
|torch.float16|256       |64             |16     |512      |823.962                 |583.107                  |1.413                         |0.453  |
|torch.float16|256       |64             |16     |512      |826.614                 |574.976                  |1.438                         |0.522  |
|torch.float16|256       |64             |16     |512      |825.938                 |578.294                  |1.428                         |0.564  |
|torch.float16|256       |64             |16     |512      |828.139                 |577.249                  |1.435                         |0.596  |
|torch.float16|256       |64             |16     |512      |826.941                 |577.976                  |1.431                         |0.622  |
|torch.float16|256       |64             |16     |512      |828.733                 |591.104                  |1.402                         |0.634  |
|torch.float16|256       |64             |16     |512      |828.672                 |583.219                  |1.421                         |0.661  |
|torch.float16|256       |64             |16     |512      |830.331                 |574.607                  |1.445                         |0.683  |
|torch.float16|256       |64             |16     |512      |829.594                 |584.991                  |1.418                         |0.701  |
|torch.float16|256       |64             |16     |512      |831.375                 |583.946                  |1.424                         |0.719  |
|torch.float16|256       |64             |16     |512      |831.304                 |581.233                  |1.430                         |0.734  |
|torch.float16|256       |64             |16     |512      |832.492                 |575.427                  |1.447                         |0.748  |
|torch.float16|256       |64             |8      |1024     |1496.945                |805.960                  |1.857                         |0.041  |
|torch.float16|256       |64             |8      |1024     |1492.695                |731.843                  |2.040                         |0.201  |
|torch.float16|256       |64             |8      |1024     |1483.008                |685.619                  |2.163                         |0.306  |
|torch.float16|256       |64             |8      |1024     |1483.807                |657.347                  |2.257                         |0.390  |
|torch.float16|256       |64             |8      |1024     |1485.773                |628.705                  |2.363                         |0.453  |
|torch.float16|256       |64             |8      |1024     |1482.783                |608.338                  |2.437                         |0.522  |
|torch.float16|256       |64             |8      |1024     |1482.127                |597.944                  |2.479                         |0.564  |
|torch.float16|256       |64             |8      |1024     |1485.169                |593.203                  |2.504                         |0.596  |
|torch.float16|256       |64             |8      |1024     |1485.507                |591.616                  |2.511                         |0.622  |
|torch.float16|256       |64             |8      |1024     |1484.360                |596.244                  |2.490                         |0.634  |
|torch.float16|256       |64             |8      |1024     |1484.268                |587.520                  |2.526                         |0.661  |
|torch.float16|256       |64             |8      |1024     |1484.319                |596.337                  |2.489                         |0.683  |
|torch.float16|256       |64             |8      |1024     |1486.797                |582.236                  |2.554                         |0.701  |
|torch.float16|256       |64             |8      |1024     |1484.534                |587.848                  |2.525                         |0.719  |
|torch.float16|256       |64             |8      |1024     |1485.855                |590.346                  |2.517                         |0.734  |
|torch.float16|256       |64             |8      |1024     |1485.046                |588.186                  |2.525                         |0.748  |
|torch.float16|256       |64             |16     |1024     |1597.942                |799.887                  |1.998                         |0.041  |
|torch.float16|256       |64             |16     |1024     |1591.429                |731.556                  |2.175                         |0.201  |
|torch.float16|256       |64             |16     |1024     |1584.261                |691.446                  |2.291                         |0.306  |
|torch.float16|256       |64             |16     |1024     |1586.299                |658.135                  |2.410                         |0.390  |
|torch.float16|256       |64             |16     |1024     |1588.675                |630.190                  |2.521                         |0.453  |
|torch.float16|256       |64             |16     |1024     |1587.139                |609.444                  |2.604                         |0.522  |
|torch.float16|256       |64             |16     |1024     |1588.449                |600.566                  |2.645                         |0.564  |
|torch.float16|256       |64             |16     |1024     |1590.374                |599.931                  |2.651                         |0.596  |
|torch.float16|256       |64             |16     |1024     |1590.354                |596.541                  |2.666                         |0.622  |
|torch.float16|256       |64             |16     |1024     |1589.565                |594.217                  |2.675                         |0.634  |
|torch.float16|256       |64             |16     |1024     |1590.293                |591.739                  |2.687                         |0.661  |
|torch.float16|256       |64             |16     |1024     |1590.446                |593.531                  |2.680                         |0.683  |
|torch.float16|256       |64             |16     |1024     |1593.149                |597.658                  |2.666                         |0.701  |
|torch.float16|256       |64             |16     |1024     |1591.511                |595.855                  |2.671                         |0.719  |
|torch.float16|256       |64             |16     |1024     |1592.361                |587.592                  |2.710                         |0.734  |
|torch.float16|256       |64             |16     |1024     |1592.095                |602.911                  |2.641                         |0.748  |
|torch.float16|256       |128            |8      |512      |1507.103                |643.113                  |2.343                         |0.041  |
|torch.float16|256       |128            |8      |512      |1512.120                |617.841                  |2.447                         |0.199  |
|torch.float16|256       |128            |8      |512      |1514.721                |650.138                  |2.330                         |0.304  |
|torch.float16|256       |128            |8      |512      |1518.367                |616.950                  |2.461                         |0.387  |
|torch.float16|256       |128            |8      |512      |1523.036                |602.286                  |2.529                         |0.449  |
|torch.float16|256       |128            |8      |512      |1521.879                |594.074                  |2.562                         |0.519  |
|torch.float16|256       |128            |8      |512      |1527.234                |595.210                  |2.566                         |0.561  |
|torch.float16|256       |128            |8      |512      |1525.074                |601.713                  |2.535                         |0.591  |
|torch.float16|256       |128            |8      |512      |1530.153                |650.926                  |2.351                         |0.618  |
|torch.float16|256       |128            |8      |512      |1531.853                |594.319                  |2.577                         |0.631  |
|torch.float16|256       |128            |8      |512      |1528.904                |594.145                  |2.573                         |0.656  |
|torch.float16|256       |128            |8      |512      |1530.030                |588.739                  |2.599                         |0.679  |
|torch.float16|256       |128            |8      |512      |1534.331                |586.168                  |2.618                         |0.698  |
|torch.float16|256       |128            |8      |512      |1534.966                |592.077                  |2.593                         |0.716  |
|torch.float16|256       |128            |8      |512      |1532.559                |592.906                  |2.585                         |0.731  |
|torch.float16|256       |128            |8      |512      |1536.256                |591.258                  |2.598                         |0.745  |
|torch.float16|256       |128            |16     |512      |1901.005                |632.525                  |3.005                         |0.041  |
|torch.float16|256       |128            |16     |512      |1910.016                |650.557                  |2.936                         |0.199  |
|torch.float16|256       |128            |16     |512      |1917.798                |606.054                  |3.164                         |0.304  |
|torch.float16|256       |128            |16     |512      |1922.437                |624.036                  |3.081                         |0.387  |
|torch.float16|256       |128            |16     |512      |1931.694                |597.586                  |3.232                         |0.449  |
|torch.float16|256       |128            |16     |512      |1931.510                |597.729                  |3.231                         |0.519  |
|torch.float16|256       |128            |16     |512      |1938.022                |595.108                  |3.257                         |0.561  |
|torch.float16|256       |128            |16     |512      |1938.248                |605.184                  |3.203                         |0.591  |
|torch.float16|256       |128            |16     |512      |1944.013                |595.292                  |3.266                         |0.618  |
|torch.float16|256       |128            |16     |512      |1945.334                |591.319                  |3.290                         |0.631  |
|torch.float16|256       |128            |16     |512      |1944.422                |602.307                  |3.228                         |0.656  |
|torch.float16|256       |128            |16     |512      |1946.562                |604.508                  |3.220                         |0.679  |
|torch.float16|256       |128            |16     |512      |1952.092                |640.573                  |3.047                         |0.698  |
|torch.float16|256       |128            |16     |512      |1953.260                |579.338                  |3.372                         |0.716  |
|torch.float16|256       |128            |16     |512      |1952.072                |579.328                  |3.370                         |0.731  |
|torch.float16|256       |128            |16     |512      |1957.161                |578.406                  |3.384                         |0.745  |
|torch.float16|256       |128            |8      |1024     |3009.055                |1422.930                 |2.115                         |0.041  |
|torch.float16|256       |128            |8      |1024     |3008.440                |1205.115                 |2.496                         |0.199  |
|torch.float16|256       |128            |8      |1024     |3000.586                |1101.742                 |2.723                         |0.304  |
|torch.float16|256       |128            |8      |1024     |3002.266                |944.937                  |3.177                         |0.387  |
|torch.float16|256       |128            |8      |1024     |2995.948                |928.256                  |3.228                         |0.449  |
|torch.float16|256       |128            |8      |1024     |2994.616                |818.186                  |3.660                         |0.519  |
|torch.float16|256       |128            |8      |1024     |2989.476                |780.493                  |3.830                         |0.561  |
|torch.float16|256       |128            |8      |1024     |2986.168                |738.816                  |4.042                         |0.591  |
|torch.float16|256       |128            |8      |1024     |2989.435                |719.135                  |4.157                         |0.618  |
|torch.float16|256       |128            |8      |1024     |2991.032                |715.459                  |4.181                         |0.631  |
|torch.float16|256       |128            |8      |1024     |2982.758                |688.763                  |4.331                         |0.656  |
|torch.float16|256       |128            |8      |1024     |2978.273                |673.782                  |4.420                         |0.679  |
|torch.float16|256       |128            |8      |1024     |2978.427                |654.838                  |4.548                         |0.698  |
|torch.float16|256       |128            |8      |1024     |2979.461                |672.963                  |4.427                         |0.716  |
|torch.float16|256       |128            |8      |1024     |2979.789                |637.972                  |4.671                         |0.731  |
|torch.float16|256       |128            |8      |1024     |2982.144                |627.988                  |4.749                         |0.745  |
|torch.float16|256       |128            |16     |1024     |3380.695                |1456.773                 |2.321                         |0.041  |
|torch.float16|256       |128            |16     |1024     |3386.992                |1241.334                 |2.729                         |0.199  |
|torch.float16|256       |128            |16     |1024     |3388.181                |1139.395                 |2.974                         |0.304  |
|torch.float16|256       |128            |16     |1024     |3388.774                |980.019                  |3.458                         |0.387  |
|torch.float16|256       |128            |16     |1024     |3387.279                |926.700                  |3.655                         |0.449  |
|torch.float16|256       |128            |16     |1024     |3387.699                |847.657                  |3.997                         |0.519  |
|torch.float16|256       |128            |16     |1024     |3386.133                |791.849                  |4.276                         |0.561  |
|torch.float16|256       |128            |16     |1024     |3385.948                |783.381                  |4.322                         |0.591  |
|torch.float16|256       |128            |16     |1024     |3389.676                |741.765                  |4.570                         |0.618  |
|torch.float16|256       |128            |16     |1024     |3389.747                |726.415                  |4.666                         |0.631  |
|torch.float16|256       |128            |16     |1024     |3384.914                |692.060                  |4.891                         |0.656  |
|torch.float16|256       |128            |16     |1024     |3381.740                |722.719                  |4.679                         |0.679  |
|torch.float16|256       |128            |16     |1024     |3383.614                |659.999                  |5.127                         |0.698  |
|torch.float16|256       |128            |16     |1024     |3385.610                |650.066                  |5.208                         |0.716  |
|torch.float16|256       |128            |16     |1024     |3386.327                |673.321                  |5.029                         |0.731  |
|torch.float16|256       |128            |16     |1024     |3390.495                |622.500                  |5.447                         |0.745  |
|torch.float16|256       |256            |8      |512      |3659.049                |1139.292                 |3.212                         |0.041  |
|torch.float16|256       |256            |8      |512      |3663.882                |982.006                  |3.731                         |0.199  |
|torch.float16|256       |256            |8      |512      |3687.219                |854.641                  |4.314                         |0.303  |
|torch.float16|256       |256            |8      |512      |3689.810                |762.184                  |4.841                         |0.386  |
|torch.float16|256       |256            |8      |512      |3698.248                |689.889                  |5.361                         |0.448  |
|torch.float16|256       |256            |8      |512      |3699.958                |634.542                  |5.831                         |0.518  |
|torch.float16|256       |256            |8      |512      |3703.358                |618.158                  |5.991                         |0.559  |
|torch.float16|256       |256            |8      |512      |3701.801                |637.399                  |5.808                         |0.590  |
|torch.float16|256       |256            |8      |512      |3706.327                |645.960                  |5.738                         |0.616  |
|torch.float16|256       |256            |8      |512      |3706.819                |588.995                  |6.293                         |0.629  |
|torch.float16|256       |256            |8      |512      |3710.587                |593.500                  |6.252                         |0.654  |
|torch.float16|256       |256            |8      |512      |3715.994                |598.354                  |6.210                         |0.676  |
|torch.float16|256       |256            |8      |512      |3719.434                |589.916                  |6.305                         |0.696  |
|torch.float16|256       |256            |8      |512      |3729.848                |594.022                  |6.279                         |0.713  |
|torch.float16|256       |256            |8      |512      |3727.216                |590.356                  |6.314                         |0.729  |
|torch.float16|256       |256            |8      |512      |3735.245                |595.292                  |6.275                         |0.742  |
|torch.float16|256       |256            |16     |512      |5160.233                |1454.059                 |3.549                         |0.041  |
|torch.float16|256       |256            |16     |512      |5193.646                |1252.291                 |4.147                         |0.199  |
|torch.float16|256       |256            |16     |512      |5215.007                |1122.140                 |4.647                         |0.303  |
|torch.float16|256       |256            |16     |512      |5232.661                |967.506                  |5.408                         |0.386  |
|torch.float16|256       |256            |16     |512      |5248.348                |871.782                  |6.020                         |0.448  |
|torch.float16|256       |256            |16     |512      |5264.804                |772.393                  |6.816                         |0.518  |
|torch.float16|256       |256            |16     |512      |5277.891                |698.255                  |7.559                         |0.559  |
|torch.float16|256       |256            |16     |512      |5283.102                |658.115                  |8.028                         |0.590  |
|torch.float16|256       |256            |16     |512      |5289.359                |625.551                  |8.456                         |0.616  |
|torch.float16|256       |256            |16     |512      |5294.019                |616.172                  |8.592                         |0.629  |
|torch.float16|256       |256            |16     |512      |5300.798                |643.768                  |8.234                         |0.654  |
|torch.float16|256       |256            |16     |512      |5310.136                |592.200                  |8.967                         |0.676  |
|torch.float16|256       |256            |16     |512      |5318.082                |607.631                  |8.752                         |0.696  |
|torch.float16|256       |256            |16     |512      |5332.429                |581.693                  |9.167                         |0.713  |
|torch.float16|256       |256            |16     |512      |5332.429                |592.353                  |9.002                         |0.729  |
|torch.float16|256       |256            |16     |512      |5343.355                |603.105                  |8.860                         |0.742  |
|torch.float16|256       |256            |8      |1024     |6646.201                |3111.977                 |2.136                         |0.041  |
|torch.float16|256       |256            |8      |1024     |6670.848                |2631.577                 |2.535                         |0.199  |
|torch.float16|256       |256            |8      |1024     |6669.005                |2291.630                 |2.910                         |0.303  |
|torch.float16|256       |256            |8      |1024     |6672.302                |2002.411                 |3.332                         |0.386  |
|torch.float16|256       |256            |8      |1024     |6670.971                |1815.060                 |3.675                         |0.448  |
|torch.float16|256       |256            |8      |1024     |6677.975                |1587.149                 |4.208                         |0.518  |
|torch.float16|256       |256            |8      |1024     |6681.733                |1448.622                 |4.612                         |0.559  |
|torch.float16|256       |256            |8      |1024     |6676.470                |1342.280                 |4.974                         |0.590  |
|torch.float16|256       |256            |8      |1024     |6678.313                |1244.006                 |5.368                         |0.616  |
|torch.float16|256       |256            |8      |1024     |6684.406                |1245.471                 |5.367                         |0.629  |
|torch.float16|256       |256            |8      |1024     |6682.296                |1164.534                 |5.738                         |0.654  |
|torch.float16|256       |256            |8      |1024     |6683.239                |1083.924                 |6.166                         |0.676  |
|torch.float16|256       |256            |8      |1024     |6689.781                |981.627                  |6.815                         |0.696  |
|torch.float16|256       |256            |8      |1024     |6688.174                |965.591                  |6.927                         |0.713  |
|torch.float16|256       |256            |8      |1024     |6692.720                |912.056                  |7.338                         |0.729  |
|torch.float16|256       |256            |8      |1024     |6689.331                |848.108                  |7.887                         |0.742  |
|torch.float16|256       |256            |16     |1024     |8162.007                |3246.213                 |2.514                         |0.041  |
|torch.float16|256       |256            |16     |1024     |8197.192                |2725.826                 |3.007                         |0.199  |
|torch.float16|256       |256            |16     |1024     |8211.180                |2376.673                 |3.455                         |0.303  |
|torch.float16|256       |256            |16     |1024     |8220.754                |2081.526                 |3.949                         |0.386  |
|torch.float16|256       |256            |16     |1024     |8225.598                |1871.442                 |4.395                         |0.448  |
|torch.float16|256       |256            |16     |1024     |8255.406                |1647.165                 |5.012                         |0.518  |
|torch.float16|256       |256            |16     |1024     |8264.407                |1496.330                 |5.523                         |0.559  |
|torch.float16|256       |256            |16     |1024     |8265.636                |1377.270                 |6.001                         |0.590  |
|torch.float16|256       |256            |16     |1024     |8270.203                |1314.130                 |6.293                         |0.616  |
|torch.float16|256       |256            |16     |1024     |8274.401                |1287.332                 |6.428                         |0.629  |
|torch.float16|256       |256            |16     |1024     |8278.712                |1210.040                 |6.842                         |0.654  |
|torch.float16|256       |256            |16     |1024     |8286.218                |1122.488                 |7.382                         |0.676  |
|torch.float16|256       |256            |16     |1024     |8292.546                |1024.010                 |8.098                         |0.696  |
|torch.float16|256       |256            |16     |1024     |8302.326                |1003.540                 |8.273                         |0.713  |
|torch.float16|256       |256            |16     |1024     |8304.363                |949.770                  |8.744                         |0.729  |
|torch.float16|256       |256            |16     |1024     |8307.190                |881.633                  |9.423                         |0.742  |
|torch.float16|256       |512            |8      |512      |10362.900               |2965.023                 |3.495                         |0.041  |
|torch.float16|256       |512            |8      |512      |10449.182               |2460.426                 |4.247                         |0.199  |
|torch.float16|256       |512            |8      |512      |10498.785               |2099.507                 |5.001                         |0.303  |
|torch.float16|256       |512            |8      |512      |10530.918               |1828.250                 |5.760                         |0.385  |
|torch.float16|256       |512            |8      |512      |10560.389               |1601.096                 |6.596                         |0.447  |
|torch.float16|256       |512            |8      |512      |10615.378               |1376.440                 |7.712                         |0.516  |
|torch.float16|256       |512            |8      |512      |10626.826               |1230.326                 |8.637                         |0.558  |
|torch.float16|256       |512            |8      |512      |10648.433               |1145.856                 |9.293                         |0.589  |
|torch.float16|256       |512            |8      |512      |10663.865               |1053.778                 |10.120                        |0.615  |
|torch.float16|256       |512            |8      |512      |10674.033               |1008.456                 |10.585                        |0.628  |
|torch.float16|256       |512            |8      |512      |10689.382               |936.069                  |11.419                        |0.653  |
|torch.float16|256       |512            |8      |512      |10701.517               |865.004                  |12.372                        |0.675  |
|torch.float16|256       |512            |8      |512      |10716.508               |814.193                  |13.162                        |0.695  |
|torch.float16|256       |512            |8      |512      |10728.438               |746.629                  |14.369                        |0.712  |
|torch.float16|256       |512            |8      |512      |10732.810               |711.444                  |15.086                        |0.727  |
|torch.float16|256       |512            |8      |512      |10748.846               |686.551                  |15.656                        |0.741  |
|torch.float16|256       |512            |16     |512      |16444.580               |4112.701                 |3.998                         |0.041  |
|torch.float16|256       |512            |16     |512      |16587.325               |3398.932                 |4.880                         |0.199  |
|torch.float16|256       |512            |16     |512      |16685.414               |2911.682                 |5.731                         |0.303  |
|torch.float16|256       |512            |16     |512      |16764.324               |2494.966                 |6.719                         |0.385  |
|torch.float16|256       |512            |16     |512      |16838.441               |2188.380                 |7.694                         |0.447  |
|torch.float16|256       |512            |16     |512      |16938.741               |1862.451                 |9.095                         |0.516  |
|torch.float16|256       |512            |16     |512      |16979.691               |1659.924                 |10.229                        |0.558  |
|torch.float16|256       |512            |16     |512      |17019.637               |1526.221                 |11.151                        |0.589  |
|torch.float16|256       |512            |16     |512      |17045.964               |1397.903                 |12.194                        |0.615  |
|torch.float16|256       |512            |16     |512      |17066.261               |1336.248                 |12.772                        |0.628  |
|torch.float16|256       |512            |16     |512      |17101.322               |1237.238                 |13.822                        |0.653  |
|torch.float16|256       |512            |16     |512      |17140.869               |1124.280                 |15.246                        |0.675  |
|torch.float16|256       |512            |16     |512      |17170.616               |1062.656                 |16.158                        |0.695  |
|torch.float16|256       |512            |16     |512      |17198.623               |973.937                  |17.659                        |0.712  |
|torch.float16|256       |512            |16     |512      |17222.461               |920.095                  |18.718                        |0.727  |
|torch.float16|256       |512            |16     |512      |17242.676               |879.503                  |19.605                        |0.741  |
|torch.float16|256       |512            |8      |1024     |17628.560               |7482.081                 |2.356                         |0.041  |
|torch.float16|256       |512            |8      |1024     |17714.146               |6213.130                 |2.851                         |0.199  |
|torch.float16|256       |512            |8      |1024     |17750.026               |5350.502                 |3.317                         |0.303  |
|torch.float16|256       |512            |8      |1024     |17786.409               |4634.020                 |3.838                         |0.385  |
|torch.float16|256       |512            |8      |1024     |17813.842               |4108.892                 |4.335                         |0.447  |
|torch.float16|256       |512            |8      |1024     |17843.066               |3547.689                 |5.029                         |0.516  |
|torch.float16|256       |512            |8      |1024     |17860.281               |3200.031                 |5.581                         |0.558  |
|torch.float16|256       |512            |8      |1024     |17883.628               |2946.642                 |6.069                         |0.589  |
|torch.float16|256       |512            |8      |1024     |17897.780               |2707.866                 |6.610                         |0.615  |
|torch.float16|256       |512            |8      |1024     |17910.763               |2604.954                 |6.876                         |0.628  |
|torch.float16|256       |512            |8      |1024     |17919.478               |2414.213                 |7.422                         |0.653  |
|torch.float16|256       |512            |8      |1024     |17937.971               |2239.283                 |8.011                         |0.675  |
|torch.float16|256       |512            |8      |1024     |17949.185               |2088.438                 |8.595                         |0.695  |
|torch.float16|256       |512            |8      |1024     |17961.923               |1955.584                 |9.185                         |0.712  |
|torch.float16|256       |512            |8      |1024     |17971.508               |1845.422                 |9.738                         |0.727  |
|torch.float16|256       |512            |8      |1024     |17985.321               |1750.456                 |10.275                        |0.741  |
|torch.float16|256       |512            |16     |1024     |23706.133               |7915.591                 |2.995                         |0.041  |
|torch.float16|256       |512            |16     |1024     |23857.368               |6554.388                 |3.640                         |0.199  |
|torch.float16|256       |512            |16     |1024     |23963.884               |5646.234                 |4.244                         |0.303  |
|torch.float16|256       |512            |16     |1024     |24047.595               |4873.390                 |4.934                         |0.385  |
|torch.float16|256       |512            |16     |1024     |24123.391               |4334.664                 |5.565                         |0.447  |
|torch.float16|256       |512            |16     |1024     |24205.149               |3720.817                 |6.505                         |0.516  |
|torch.float16|256       |512            |16     |1024     |24254.690               |3343.667                 |7.254                         |0.558  |
|torch.float16|256       |512            |16     |1024     |24294.893               |3081.001                 |7.885                         |0.589  |
|torch.float16|256       |512            |16     |1024     |24318.115               |2864.405                 |8.490                         |0.615  |
|torch.float16|256       |512            |16     |1024     |24338.862               |2728.325                 |8.921                         |0.628  |
|torch.float16|256       |512            |16     |1024     |24372.214               |2525.000                 |9.652                         |0.653  |
|torch.float16|256       |512            |16     |1024     |24412.764               |2337.894                 |10.442                        |0.675  |
|torch.float16|256       |512            |16     |1024     |24445.256               |2188.349                 |11.171                        |0.695  |
|torch.float16|256       |512            |16     |1024     |24472.791               |2041.252                 |11.989                        |0.712  |
|torch.float16|256       |512            |16     |1024     |24497.439               |1923.656                 |12.735                        |0.727  |
|torch.float16|256       |512            |16     |1024     |24518.420               |1823.457                 |13.446                        |0.741  |